msgid ""
msgstr ""
"Project-Id-Version: notmusicbot\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-11-08 07:52-0800\n"
"PO-Revision-Date: 2024-11-08 18:12\n"
"Last-Translator: \n"
"Language-Team: Italian\n"
"Language: it_IT\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generated-By: pygettext.py 1.5-mb01\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Crowdin-Project: notmusicbot\n"
"X-Crowdin-Project-ID: 734017\n"
"X-Crowdin-Language: it\n"
"X-Crowdin-File: /fae/i18n/musicbot_logs.pot\n"
"X-Crowdin-File-ID: 26\n"

#: musicbot/aliases.py:52
msgid "Aliases file not found, copying example_aliases.json"
msgstr "File alias non trovato, copia di example_aliases.json"

#: musicbot/aliases.py:56
msgid "Error while loading aliases.\n\n"
"Problem:\n"
"  Your aliases files (aliases.json & example_aliases.json) are missing.\n\n"
"Solution:\n"
"  Replace the alias config file(s) or copy them from:\n"
"    https://github.com/Just-Some-Bots/MusicBot/"
msgstr "Errore durante il caricamento delle alias.\n\n"
"Problema:\n"
"  Mancano i tuoi file alias (aliases.json & example_aliases.json). Soluzione\n\n"
":\n"
"  Sostituire il file di configurazione alias o copiarlo da:\n"
"    https://github.com/Just-Some-Bots/MusicBot/"

#: musicbot/aliases.py:80
#, python-format
msgid "Failed to load aliases file:  %s"
msgstr "Caricamento del file alias non riuscito:  %s"

#: musicbot/aliases.py:88
#, python-format
msgid "Failed to parse aliases file:  %s\n"
"Ensure the file contains valid JSON and restart the bot."
msgstr "Analisi del file alias non riuscita:  %s\n"
"Assicurarsi che il file contenga JSON valido e riavviare il bot."

#: musicbot/aliases.py:114
#, python-format
msgid "Aliases skipped for non-existent command:  %(command)s  ->  %(aliases)s"
msgstr "Alias saltati per il comando inesistente:  %(command)s  ->  %(aliases)s"

#: musicbot/aliases.py:122
#, python-format
msgid "Alias(es) skipped for invalid alias data:  %(command)s  ->  %(aliases)s"
msgstr "Alias(es) saltato per dati alias non validi:  %(command)s  ->  %(aliases)s"

#: musicbot/aliases.py:132
#, python-format
msgid "Alias `%(alias)s` skipped as already exists on command:  %(command)s"
msgstr "Alias `%(alias)s` saltato come esiste già sul comando:  %(command)s"

#: musicbot/autoplaylist.py:76
#, python-format
msgid "Error loading auto playlist file:  %s"
msgstr "Errore nel caricare il file di scaletta automatica:  %s"

#: musicbot/autoplaylist.py:121
#, python-format
msgid "Removing unplayable song from playlist, %(playlist)s: %(track)s"
msgstr "Rimozione del brano non riproducibile dalla playlist, %(playlist)s: %(track)s"

#: musicbot/autoplaylist.py:126
#, python-format
msgid "Removing song from playlist, %(playlist)s: %(track)s"
msgstr "Rimozione del brano dalla playlist, %(playlist)s: %(track)s"

#: musicbot/autoplaylist.py:151
msgid "Could not log information about the playlist URL removal."
msgstr "Impossibile registrare le informazioni sulla rimozione dell'URL della playlist."

#: musicbot/autoplaylist.py:155
msgid "Updating playlist file..."
msgstr "Aggiornamento file playlist..."

#: musicbot/autoplaylist.py:171 musicbot/autoplaylist.py:208
#, python-format
msgid "Failed to save playlist file:  %s"
msgstr "Salvataggio del file playlist non riuscito:  %s"

#: musicbot/autoplaylist.py:180
#, python-format
msgid "URL already in playlist %s, ignoring"
msgstr "URL già in playlist %s, ignorando"

#: musicbot/autoplaylist.py:187
#, python-format
msgid "Adding new URL to playlist, %(playlist)s: %(track)s"
msgstr "Aggiunta di un nuovo URL alla playlist, %(playlist)s: %(track)s"

#: musicbot/bot.py:163
#, python-format
msgid "Initializing MusicBot %s"
msgstr "Inizializzazione MusicBot %s"

#: musicbot/bot.py:243
#, python-format
msgid "Loop is closed, cannot create task for: %r"
msgstr "Loop è chiuso, impossibile creare task per: %r"

#: musicbot/bot.py:259
#, python-format
msgid "Unhandled exception for task:  %r"
msgstr "Eccezione non gestita per l'attività:  %r"

#: musicbot/bot.py:263
#, python-format
msgid "Unhandled exception for task:  %(task)r  --  %(raw_error)s"
msgstr "Eccezione non gestita per l'attività:  %(task)r --  %(raw_error)s"

#: musicbot/bot.py:308 musicbot/bot.py:330
msgid "Spotify did not provide us with a token. Disabling."
msgstr "Spotify non ci ha fornito un token. Disattivazione."

#: musicbot/bot.py:312
msgid "Authenticated with Spotify successfully using client ID and secret."
msgstr "Autenticato con Spotify con successo utilizzando l'ID client e segreto."

#: musicbot/bot.py:316
#, python-format
msgid "Could not start Spotify client. Is your client ID and secret correct? Details: %s. Continuing anyway in 5 seconds..."
msgstr "Impossibile avviare il client Spotify. Il tuo ID client e segreto sono corretti? Dettagli: %s. Continuare comunque in 5 secondi..."

#: musicbot/bot.py:324
msgid "The config did not have Spotify app credentials, attempting to use guest mode."
msgstr "La configurazione non ha le credenziali dell'applicazione Spotify, tentando di utilizzare la modalità ospite."

#: musicbot/bot.py:334
msgid "Authenticated with Spotify successfully using guest mode."
msgstr "Autenticato con Spotify con successo utilizzando la modalità ospite."

#: musicbot/bot.py:339
#, python-format
msgid "Could not start Spotify client using guest mode. Details: %s."
msgstr "Impossibile avviare il client Spotify usando la modalità ospite. Dettagli: %s."

#: musicbot/bot.py:347
msgid "Experimental Yt-dlp OAuth2 plugin is enabled. This might break at any point!"
msgstr "Il plugin sperimentale Yt-dlp OAuth2 è abilitato. Questo potrebbe rompere in qualsiasi punto!"

#: musicbot/bot.py:354
msgid "Initialized, now connecting to discord."
msgstr "Inizializzato, ora si collega alla discordia."

#: musicbot/bot.py:365
msgid "Network ping test is disabled via config."
msgstr "Il test di rete ping è disabilitato tramite config."

#: musicbot/bot.py:369
msgid "Network ping test is closing down."
msgstr "Test ping di rete si sta chiudendo."

#: musicbot/bot.py:380
msgid "Could not resolve ping target."
msgstr "Impossibile risolvere il ping target."

#: musicbot/bot.py:410
msgid "Network ping test cancelled."
msgstr "Test ping di rete annullato."

#: musicbot/bot.py:424
msgid "Network testing via HTTP does not have a session to borrow."
msgstr "Il test di rete tramite HTTP non ha una sessione in prestito."

#: musicbot/bot.py:449
msgid "Could not locate `ping` executable in your environment."
msgstr "Impossibile trovare l'eseguibile `ping` nel tuo ambiente."

#: musicbot/bot.py:475
#, python-format
msgid "MusicBot could not locate a `ping` command path.  Will attempt to use HTTP ping instead.\n"
"MusicBot tried the following command:   %s\n"
"You should enable ping in your system or container environment for best results.\n"
"Alternatively disable network checking via config."
msgstr "MusicBot non può trovare un percorso di comando `ping`. Tenterà invece di usare il ping HTTP.\n"
"MusicBot ha provato il seguente comando:   %s\n"
"Dovresti abilitare il ping nel tuo ambiente di sistema o contenitore per ottenere i migliori risultati.\n"
"In alternativa disabilita il controllo di rete tramite config."

#: musicbot/bot.py:485
#, python-format
msgid "MusicBot was denied permission to execute the `ping` command.  Will attempt to use HTTP ping instead.\n"
"MusicBot tried the following command:   %s\n"
"You should enable ping in your system or container environment for best results.\n"
"Alternatively disable network checking via config."
msgstr "MusicBot è stato negato il permesso di eseguire il comando `ping`. Tenterà invece di usare il ping HTTP.\n"
"MusicBot ha provato il seguente comando:   %s\n"
"Dovresti abilitare il ping nel tuo ambiente di sistema o contenitore per ottenere i migliori risultati.\n"
"In alternativa disabilita il controllo di rete tramite config."

#: musicbot/bot.py:495
#, python-format
msgid "Your environment may not allow the `ping` system command.  Will attempt to use HTTP ping instead.\n"
"MusicBot tried the following command:   %s\n"
"You should enable ping in your system or container environment for best results.\n"
"Alternatively disable network checking via config."
msgstr "Il tuo ambiente potrebbe non consentire il comando di sistema `ping`. Tenterà invece di usare il ping HTTP.\n"
"MusicBot ha provato il seguente comando:   %s\n"
"Dovresti abilitare il ping nel tuo ambiente di sistema o contenitore per ottenere i migliori risultati.\n"
"In alternativa disabilita il controllo di rete tramite config."

#: musicbot/bot.py:510
msgid "MusicBot detected network is available again."
msgstr "MusicBot rilevato rete è nuovamente disponibile."

#: musicbot/bot.py:515
msgid "VoiceClient is not connected, waiting to resume MusicPlayer..."
msgstr "VoiceClient non è connesso, in attesa di riprendere MusicPlayer..."

#: musicbot/bot.py:519
#, python-format
msgid "Resuming playback of player:  (%(guild_id)s) %(player)r"
msgstr "Riprendendo la riproduzione del giocatore: (%(guild_id)s) %(player)r"

#: musicbot/bot.py:530
msgid "MusicBot detected a network outage."
msgstr "MusicBot ha rilevato un'interruzione di rete."

#: musicbot/bot.py:534
#, python-format
msgid "Pausing MusicPlayer due to network availability:  (%(guild_id)s) %(player)r"
msgstr "Pausa MusicPlayer a causa della disponibilità della rete: (%(guild_id)s) %(player)r"

#: musicbot/bot.py:555
#, python-format
msgid "Looking for owner in guild: %(guild)s (required voice: %(required)s) and got:  %(owner)s"
msgstr "Alla ricerca del proprietario nella gilda: %(guild)s (voce obbligatoria: %(required)s) e ottiene:  %(owner)s"

#: musicbot/bot.py:569
msgid "Checking for channels to auto-join or resume..."
msgstr "Controllo della presenza di canali da auto-entrare o riprendere..."

#: musicbot/bot.py:581
#, python-format
msgid "Guild not available, cannot auto join:  %(id)s/%(name)s"
msgstr "Gilda non disponibile, impossibile entrare automaticamente:  %(id)s/%(name)s"

#: musicbot/bot.py:589
#, python-format
msgid "Found resumable voice channel:  %(channel)s  in guild:  %(guild)s"
msgstr "Trovato canale vocale riproducibile:  %(channel)s  nella gilda:  %(guild)s"

#: musicbot/bot.py:602
#, python-format
msgid "Will try resuming voice session instead of Auto-Joining channel:  %s"
msgstr "Proverà a riprendere la sessione vocale invece di entrare automaticamente nel canale:  %s"

#: musicbot/bot.py:619
#, python-format
msgid "Found owner in voice channel:  %s"
msgstr "Trovato proprietario nel canale vocale:  %s"

#: musicbot/bot.py:624
#, python-format
msgid "Ignoring resumable channel, AutoSummon to owner in channel:  %s"
msgstr "Ignorando il canale riattivabile, AutoSummon al proprietario nel canale:  %s"

#: musicbot/bot.py:629
#, python-format
msgid "Ignoring Auto-Join channel, AutoSummon to owner in channel:  %s"
msgstr "Ignorando il canale Auto-Entra automatico, AutoSummon al proprietario nel canale:  %s"

#: musicbot/bot.py:641
#, python-format
msgid "Already connected to channel:  %(channel)s  in guild:  %(guild)s"
msgstr "Già connesso al canale:  %(channel)s  nella gilda:  %(guild)s"

#: musicbot/bot.py:650
#, python-format
msgid "Attempting to join channel:  %(channel)s  in guild:  %(guild)s"
msgstr "Tentativo di entrare nel canale:  %(channel)s  nella gilda:  %(guild)s"

#: musicbot/bot.py:657
msgid "Discarding MusicPlayer and making a new one..."
msgstr "Scartare MusicPlayer e crearne uno nuovo..."

#: musicbot/bot.py:677
msgid "MusicBot will make a new MusicPlayer now..."
msgstr "MusicBot renderà un nuovo MusicPlayer ora..."

#: musicbot/bot.py:698
#, python-format
msgid "Not joining %(guild)s/%(channel)s, it isn't a supported voice channel."
msgstr "Non entrando in %(guild)s/%(channel)s, non è un canale vocale supportato."

#: musicbot/bot.py:701
msgid "Finished joining configured channels."
msgstr "Completato il collegamento ai canali configurati."

#: musicbot/bot.py:713
msgid "Member is not voice-enabled and cannot use this command."
msgstr "Il membro non è abilitato alla voce e non può usare questo comando."

#: musicbot/bot.py:721
msgid "You cannot use this command when not in the voice channel."
msgstr "Non puoi usare questo comando quando non è nel canale vocale."

#: musicbot/bot.py:735
msgid "Getting bot Application Info."
msgstr "Ottenere bot Application Info."

#: musicbot/bot.py:760
#, python-format
msgid "MusicBot does not have permission to Connect in channel:  %s"
msgstr "MusicBot non ha il permesso di connettersi nel canale:  %s"

#: musicbot/bot.py:764
#, python-format
msgid "MusicBot does not have permission to Connect in channel:  `%(name)s`"
msgstr "MusicBot non ha il permesso di connettersi nel canale: `%(name)s`"

#: musicbot/bot.py:769
#, python-format
msgid "MusicBot does not have permission to Speak in channel:  %s"
msgstr "MusicBot non ha il permesso di parlare nel canale:  %s"

#: musicbot/bot.py:773
#, python-format
msgid "MusicBot does not have permission to Speak in channel:  `%(name)s`"
msgstr "MusicBot non ha il permesso di parlare nel canale: `%(name)s`"

#: musicbot/bot.py:783
#, python-format
msgid "Reusing bots VoiceClient from guild:  %s"
msgstr "Riutilizzo di bot VoiceClient dalla gilda:  %s"

#: musicbot/bot.py:788
#, python-format
msgid "Forcing disconnect on stale VoiceClient in guild:  %s"
msgstr "Forzare la disconnessione su VoiceClient obsoleto nella gilda:  %s"

#: musicbot/bot.py:794
msgid "Disconnect failed or was cancelled?"
msgstr "Disconnessione fallita o annullata?"

#: musicbot/bot.py:802
#, python-format
msgid "MusicBot is unable to connect to the channel right now:  %(channel)s"
msgstr "MusicBot non è in grado di connettersi al canale in questo momento:  %(channel)s"

#: musicbot/bot.py:806
msgid "MusicBot could not connect to the channel.\n"
"Try again later, or restart the bot if this continues."
msgstr "MusicBot non può connettersi al canale.\n"
"Riprova più tardi, o riavvia il bot se continua."

#: musicbot/bot.py:817
msgid "MusicBot has a VoiceClient now..."
msgstr "MusicBot ha un VoiceClient ora..."

#: musicbot/bot.py:822
#, python-format
msgid "Retrying connection after a timeout error (%(attempt)s) while trying to connect to:  %(channel)s"
msgstr "Riprovare la connessione dopo un errore di timeout (%(attempt)s) durante il tentativo di connettersi a:  %(channel)s"

#: musicbot/bot.py:827
msgid "MusicBot VoiceClient connection attempt was cancelled. No retry."
msgstr "Il tentativo di connessione MusicBot VoiceClient è stato annullato. Nessun riprovo."

#: musicbot/bot.py:830
msgid "MusicBot connection to voice was cancelled. This is odd. Maybe restart?"
msgstr "La connessione MusicBot alla voce è stata annullata. Questo è strano. Forse riavviare?"

#: musicbot/bot.py:840 musicbot/bot.py:8386
msgid "MusicBot does not have permission to speak."
msgstr "MusicBot non ha il permesso di parlare."

#: musicbot/bot.py:844 musicbot/bot.py:8388
msgid "MusicBot could not request to speak."
msgstr "MusicBot non può richiedere di parlare."

#: musicbot/bot.py:857
#, python-format
msgid "Disconnecting a MusicPlayer in guild:  %s"
msgstr "Disconnettere un musicista nella gilda:  %s"

#: musicbot/bot.py:869
msgid "Disconnecting VoiceClient before we kill the MusicPlayer."
msgstr "Scollegare VoiceClient prima di uccidere MusicPlayer."

#: musicbot/bot.py:877 musicbot/bot.py:896 musicbot/bot.py:908
#: musicbot/bot.py:931
msgid "The disconnect failed or was cancelled."
msgstr "La disconnessione non è riuscita o è stata annullata."

#: musicbot/bot.py:887
msgid "MusicBot has a VoiceProtocol that is not a VoiceClient. Disconnecting anyway..."
msgstr "MusicBot ha un VoiceProtocol che non è un VoiceClient. Disconnettersi comunque..."

#: musicbot/bot.py:900
#, python-format
msgid "Disconnecting a rogue VoiceClient in guild:  %s"
msgstr "Disconnettere un VoiceClient canaglia nella gilda:  %s"

#: musicbot/bot.py:924
msgid "Disconnecting a non-guild VoiceClient..."
msgstr "Disconnettere un non-guild VoiceClient..."

#: musicbot/bot.py:934
#, python-format
msgid "MusicBot.voice_clients list contains a non-VoiceClient object?\n"
"The object is actually of type:  %s"
msgstr "La lista MusicBot.voice_clients contiene un oggetto non-VoiceClient?\n"
"L'oggetto è in realtà di tipo:  %s"

#: musicbot/bot.py:944
#, python-format
msgid "We still have a MusicPlayer ref in guild (%(guild_id)s):  %(player)r"
msgstr "Abbiamo ancora un lettore musicale nella gilda (%(guild_id)s):  %(player)r"

#: musicbot/bot.py:957
#, python-format
msgid "Guild (%(guild)s) wants a player, optional:  %(player)r"
msgstr "Gilda (%(guild)s) vuole un giocatore, opzionale:  %(player)r"

#: musicbot/bot.py:964
msgid "[BUG] MusicPlayer is missing a VoiceClient somehow.  You should probably restart the bot."
msgstr "[BUG] MusicPlayer manca un VoiceClient in qualche modo. Probabilmente dovresti riavviare il bot."

#: musicbot/bot.py:969
msgid "MusicPlayer has a VoiceClient that is not connected."
msgstr "MusicPlayer ha un VoiceClient che non è connesso."

#: musicbot/bot.py:970
#, python-format
msgid "MusicPlayer obj:  %r"
msgstr "MusicPlayer obj:  %r"

#: musicbot/bot.py:971
#, python-format
msgid "VoiceClient obj:  %r"
msgstr "VoiceClient obj:  %r"

#: musicbot/bot.py:991
#, python-format
msgid "Getting a MusicPlayer for guild:  %(guild)s  In Channel:  %(channel)s  Create: %(create)s  Deserialize:  %(serial)s"
msgstr "Ottenere un MusicPlayer per la gilda:  %(guild)s  Nel Canale:  %(channel)s  Crea: %(create)s  Deserializza:  %(serial)s"

#: musicbot/bot.py:1007
#, python-format
msgid "Created player via deserialization for guild %(guild_id)s with %(number)s entries"
msgstr "Giocatore creato tramite deserializzazione per la gilda %(guild_id)s con voci %(number)s"

#: musicbot/bot.py:1016
msgid "The bot is not in a voice channel.\n"
"Use the summon command to bring the bot to your voice channel."
msgstr "Il bot non è in un canale vocale.\n"
"Usa il comando summon per portare il bot sul tuo canale vocale."

#: musicbot/bot.py:1028
msgid "Something is wrong, we didn't get the VoiceClient."
msgstr "Qualcosa è sbagliato, non abbiamo ottenuto il VoiceClient."

#: musicbot/bot.py:1061
msgid "Running on_player_play"
msgstr "Esecuzione on_player_play"

#: musicbot/bot.py:1136
#, python-format
msgid "Setting URL history guild %(guild_id)s == %(url)s"
msgstr "Impostazione della gilda cronologia URL %(guild_id)s == %(url)s"

#: musicbot/bot.py:1172 musicbot/bot.py:4806
#, python-format
msgid "No thumbnail set for entry with URL: %s"
msgstr "Nessuna miniatura impostata per la voce con URL: %s"

#: musicbot/bot.py:1184
msgid "no channel to put now playing message into"
msgstr "nessun canale in cui inserire il messaggio in riproduzione"

#: musicbot/bot.py:1196
msgid "ignored now-playing message as it was already posted."
msgstr "ignorato il messaggio in riproduzione ora come era già stato pubblicato."

#: musicbot/bot.py:1220
msgid "Running on_player_resume"
msgstr "Esecuzione on_player_resume"

#: musicbot/bot.py:1233
msgid "Running on_player_pause"
msgstr "Esecuzione on_player_pause"

#: musicbot/bot.py:1249
msgid "Running on_player_stop"
msgstr "Esecuzione on_player_stop"

#: musicbot/bot.py:1259
msgid "Running on_player_finished_playing"
msgstr "Esecuzione on_player_finished_playing"

#: musicbot/bot.py:1261 musicbot/bot.py:1307 musicbot/bot.py:1378
msgid "Event loop is closed, nothing else to do here."
msgstr "Il ciclo di eventi è chiuso, nient'altro da fare qui."

#: musicbot/bot.py:1265 musicbot/bot.py:1311 musicbot/bot.py:1382
msgid "Logout under way, ignoring this event."
msgstr "Logout in corso, ignorando questo evento."

#: musicbot/bot.py:1282
msgid "VoiceClient says it is not connected, nothing else we can do here."
msgstr "VoiceClient dice che non è connesso, niente altro che possiamo fare qui."

#: musicbot/bot.py:1289
msgid "Player finished and queue is empty, leaving voice channel..."
msgstr "Il giocatore finito e la coda è vuota, lasciando il canale vocale..."

#: musicbot/bot.py:1303
msgid "Looping over queue to expunge songs with missing author..."
msgstr "Caricando sopra la coda per ripulire le canzoni con l'autore mancante..."

#: musicbot/bot.py:1342
#, python-format
msgid "Author `%(user)s` absent, skipped (deleted) entry from queue:  %(song)s"
msgstr "Autore `%(user)s` assente, saltato (cancellato) dalla coda:  %(song)s"

#: musicbot/bot.py:1361
msgid "No playable songs in the Guild autoplaylist, disabling."
msgstr "Nessun brano riproducibile nella lista di riproduzione automatica della gilda, disabilitata."

#: musicbot/bot.py:1366
msgid "No content in current autoplaylist. Filling with new music..."
msgstr "Nessun contenuto nella playlist automatica. Riempimento con nuova musica..."

#: musicbot/bot.py:1374
msgid "Looping over player autoplaylist..."
msgstr "Caricamento sopra la playlist del giocatore..."

#: musicbot/bot.py:1415
#, python-format
msgid "Error while processing song \"%(url)s\":  %(raw_error)s"
msgstr "Errore durante l'elaborazione del brano \"%(url)s\":  %(raw_error)s"

#: musicbot/bot.py:1426
#, python-format
msgid "Error extracting song \"%(url)s\": %(raw_error)s"
msgstr "Errore nell'estrarre il brano \"%(url)s\": %(raw_error)s"

#: musicbot/bot.py:1441
msgid "MusicBot needs to stop the auto playlist extraction and bail."
msgstr "MusicBot ha bisogno di fermare l'estrazione della playlist automatica e la cauzione."

#: musicbot/bot.py:1446
msgid "MusicBot got an unhandled exception in player finished event."
msgstr "MusicBot ha ottenuto un'eccezione non gestita nell'evento finito dal giocatore."

#: musicbot/bot.py:1452
#, python-format
msgid "Expanding auto playlist with entries extracted from:  %s"
msgstr "Espandendo playlist automatica con voci estratte da:  %s"

#: musicbot/bot.py:1476
#, python-format
msgid "Error adding song from autoplaylist: %s"
msgstr "Errore nell'aggiungere il brano dalla playlist automatica: %s"

#: musicbot/bot.py:1479
msgid "Exception data for above error:"
msgstr "Eccezione di dati per l'errore sopra:"

#: musicbot/bot.py:1485
msgid "No playable songs in the autoplaylist, disabling."
msgstr "Nessun brano riproducibile nella playlist automatica, disabilitato."

#: musicbot/bot.py:1505
msgid "Running on_player_entry_added"
msgstr "Esecuzione su_player_entry_added"

#: musicbot/bot.py:1515
msgid "Automatically skipping auto-playlist entry for queued entry."
msgstr "Saltando automaticamente la voce della scaletta automatica per la voce in coda."

#: musicbot/bot.py:1535
#, python-format
msgid "MusicPlayer exception for entry: %r"
msgstr "Eccezione MusicPlayer per la voce: %r"

#: musicbot/bot.py:1541
msgid "MusicPlayer exception."
msgstr "Eccezione di MusicPlayer."

#: musicbot/bot.py:1565
#, python-format
msgid "Auto playlist track could not be played:  %r"
msgstr "Impossibile riprodurre la traccia della scaletta automatica:  %r"

#: musicbot/bot.py:1604
msgid "Logout under way, ignoring status update event."
msgstr "Logout in corso, ignorando l'aggiornamento dello stato."

#: musicbot/bot.py:1705
#, python-format
msgid "Update bot status:  %(status)s -- %(activity)r"
msgstr "Aggiorna stato bot:  %(status)s -- %(activity)r"

#: musicbot/bot.py:1733
#, python-format
msgid "Serializing queue for %s"
msgstr "Serializzazione della coda per %s"

#: musicbot/bot.py:1759
#, python-format
msgid "Deserializing queue for %s"
msgstr "Deserializzazione della coda per %s"

#: musicbot/bot.py:1777
#, python-format
msgid "Writing current song for %s"
msgstr "Scrittura del brano corrente per %s"

#: musicbot/bot.py:1802
#, python-format
msgid "Cannot send non-response object:  %r"
msgstr "Impossibile inviare l'oggetto non-risposta:  %r"

#: musicbot/bot.py:1807
msgid "[Dev Bug] Tried sending an invalid response object."
msgstr "[Dev Bug] Provato a inviare un oggetto di risposta non valido."

#: musicbot/bot.py:1835
#, python-format
msgid "sending embed to: %s"
msgstr "invio incorporato a: %s"

#: musicbot/bot.py:1838
#, python-format
msgid "sending text to: %s"
msgstr "invio del testo a: %s"

#: musicbot/bot.py:1843
#, python-format
msgid "Cannot send message to \"%s\", no permission"
msgstr "Impossibile inviare il messaggio a \"%s\", nessun permesso"

#: musicbot/bot.py:1850
#, python-format
msgid "Cannot send message to \"%s\", invalid or deleted channel"
msgstr "Impossibile inviare il messaggio a \"%s\", canale eliminato o non valido"

#: musicbot/bot.py:1858
#, python-format
msgid "Message is over the message size limit (%s)"
msgstr "Il messaggio è sopra il limite di dimensione del messaggio (%s)"

#: musicbot/bot.py:1866
msgid "Could not send private message, sending in fallback channel instead."
msgstr "Impossibile inviare un messaggio privato, inviando un canale di ripiego."

#: musicbot/bot.py:1884
#, python-format
msgid "Rate limited send message, retrying in %s seconds."
msgstr "Vota il messaggio di invio limitato, riprovando in %s secondi."

#: musicbot/bot.py:1890
#, python-format
msgid "Cancelled message retry for:  %s"
msgstr "Riprova messaggio annullato per:  %s"

#: musicbot/bot.py:1895
msgid "Rate limited send message, but cannot retry!"
msgstr "Valuta il messaggio di invio limitato, ma non riprovare!"

#: musicbot/bot.py:1901
msgid "Failed to send message in fallback channel."
msgstr "Impossibile inviare il messaggio nel canale di ripiego."

#: musicbot/bot.py:1906 musicbot/bot.py:1970 musicbot/bot.py:2040
msgid "Failed to send due to an HTTP error."
msgstr "Impossibile inviare a causa di un errore HTTP."

#: musicbot/bot.py:1932
#, python-format
msgid "Cannot delete message \"%s\", no permission"
msgstr "Impossibile eliminare il messaggio \"%s\", nessun permesso"

#: musicbot/bot.py:1937
#, python-format
msgid "Cannot delete message \"%s\", message not found"
msgstr "Impossibile eliminare il messaggio \"%s\", messaggio non trovato"

#: musicbot/bot.py:1955
#, python-format
msgid "Rate limited message delete, retrying in %s seconds."
msgstr "Valuta l'eliminazione limitata dei messaggi, riprovando in %s secondi."

#: musicbot/bot.py:1960
msgid "Rate limited message delete, but cannot retry!"
msgstr "Valuta l'eliminazione del messaggio limitata, ma non riprova!"

#: musicbot/bot.py:1963
msgid "Failed to delete message"
msgstr "Eliminazione del messaggio non riuscita"

#: musicbot/bot.py:1965
#, python-format
msgid "Got HTTPException trying to delete message: %s"
msgstr "Hai HTTPException cercando di eliminare il messaggio: %s"

#: musicbot/bot.py:2000
#, python-format
msgid "Cannot edit message \"%s\", message not found"
msgstr "Impossibile modificare il messaggio \"%s\", messaggio non trovato"

#: musicbot/bot.py:2004
msgid "Sending message instead"
msgstr "Invio messaggio invece"

#: musicbot/bot.py:2021
#, python-format
msgid "Rate limited edit message, retrying in %s seconds."
msgstr "Valuta il messaggio di modifica limitato, riprovando in %s secondi."

#: musicbot/bot.py:2027
#, python-format
msgid "Cancelled message edit for:  %s"
msgstr "Modifica messaggio annullata per:  %s"

#: musicbot/bot.py:2033
msgid "Failed to edit message"
msgstr "Modifica del messaggio non riuscita"

#: musicbot/bot.py:2035
#, python-format
msgid "Got HTTPException trying to edit message %s to: %s"
msgstr "Hai HTTPException tentato di modificare il messaggio %s a: %s"

#: musicbot/bot.py:2057
#, python-format
msgid "Cancelled delete for message (ID: %(id)s):  %(content)s"
msgstr "Eliminazione annullata per il messaggio (ID: %(id)s):  %(content)s"

#: musicbot/bot.py:2119
#, python-format
msgid "Caught a signal from the OS: %s"
msgstr "Ricevuto un segnale dal sistema operativo: %s"

#: musicbot/bot.py:2123
msgid "Disconnecting and closing down MusicBot..."
msgstr "Scollegare e chiudere MusicBot..."

#: musicbot/bot.py:2126
msgid "Exception thrown while handling interrupt signal!"
msgstr "Eccezione lanciata durante la gestione del segnale di interruzione!"

#: musicbot/bot.py:2140
msgid "MusicBot is now doing shutdown steps..."
msgstr "MusicBot sta ora facendo passi di arresto..."

#: musicbot/bot.py:2147
msgid "Failed Discord API Login!\n\n"
"Problem:\n"
"  MusicBot could not log into Discord API.\n"
"  Your Token may be incorrect or there may be an API outage.\n\n"
"Solution:\n"
"  Make sure you have the correct Token set in your config.\n"
"  Check API status at the official site: discordstatus.com"
msgstr "Accesso API Discord non riuscito!\n\n"
"Problema:\n"
"  MusicBot non è riuscito ad accedere all'API Discord.\n"
"  Il tuo token potrebbe essere errato o potrebbe esserci un'interruzione dell'API.\n\n"
"Soluzione:\n"
"  Assicurati di aver impostato il token corretto nella tua configurazione.\n"
"  Controlla lo stato dell'API sul sito ufficiale: discordstatus.com"

#: musicbot/bot.py:2161
msgid "Waiting for download threads to finish up..."
msgstr "In attesa che le discussioni di download finiscano..."

#: musicbot/bot.py:2187
#, python-format
msgid "Will wait for task:  %(name)s  (%(func)s)"
msgstr "Attenderà il compito:  %(name)s  (%(func)s)"

#: musicbot/bot.py:2194
#, python-format
msgid "Will try to cancel task:  %(name)s  (%(func)s)"
msgstr "Cercherà di annullare l'attività:  %(name)s  (%(func)s)"

#: musicbot/bot.py:2202
msgid "Awaiting pending tasks..."
msgstr "In attesa di attività..."

#: musicbot/bot.py:2208
msgid "Closing HTTP Connector."
msgstr "Chiusura Connettore HTTP."

#: musicbot/bot.py:2214
msgid "Closing aiohttp session."
msgstr "Chiusura sessione aiohttp."

#: musicbot/bot.py:2226
msgid "Logout has been called."
msgstr "Il logout è stato chiamato."

#: musicbot/bot.py:2238
#, python-format
msgid "Exception in %(event)s:\n"
"%(error)s"
msgstr "Eccezione in %(event)s:\n"
"%(error)s"

#: musicbot/bot.py:2254
#, python-format
msgid "Exception in %s"
msgstr "Eccezione in %s"

#: musicbot/bot.py:2261
msgid "MusicBot resumed a session with discord."
msgstr "MusicBot ha ripreso una sessione con discord."

#: musicbot/bot.py:2278
msgid "Finish on_ready"
msgstr "Termina su_pronto"

#: musicbot/bot.py:2285
msgid "Logged in, now getting MusicBot ready..."
msgstr "Accesso effettuato, ora ottenendo MusicBot pronto..."

#: musicbot/bot.py:2288
msgid "ClientUser is somehow none, we gotta bail..."
msgstr "ClientUser è in qualche modo nessuno, abbiamo ottenuto la bail..."

#: musicbot/bot.py:2297
#, python-format
msgid "MusicBot:  %(id)s/%(name)s#%(desc)s"
msgstr "MusicBot:  %(id)s/%(name)s#%(desc)s"

#: musicbot/bot.py:2308
#, python-format
msgid "Owner:     %(id)s/%(name)s#%(desc)s\n"
msgstr "Proprietario:     %(id)s/%(name)s#%(desc)s\n"

#: musicbot/bot.py:2316 musicbot/bot.py:2343
msgid "Guild List:"
msgstr "Lista Gilda:"

#: musicbot/bot.py:2320 musicbot/bot.py:2346 musicbot/bot.py:8572
#, python-format
msgid " - %s"
msgstr " - %s"

#: musicbot/bot.py:2329
#, python-format
msgid "Left %s due to bot owner not found"
msgstr "Sinistra %s a causa del proprietario del bot non trovato"

#: musicbot/bot.py:2334
#, python-format
msgid "Not proceeding with checks in %s servers due to unavailability"
msgstr "Non procedere con i controlli nei server %s a causa di indisponibilità"

#: musicbot/bot.py:2340
#, python-format
msgid "Owner could not be found on any guild (id: %s)\n"
msgstr "Il proprietario non può essere trovato in nessuna gilda (id: %s)\n"

#: musicbot/bot.py:2349
msgid "Owner unknown, bot is not on any guilds."
msgstr "Proprietario sconosciuto, bot non è su alcuna gilda."

#: musicbot/bot.py:2353
#, python-format
msgid "To make the bot join a guild, paste this link in your browser. \n"
"Note: You should be logged into your main account and have \n"
"manage server permissions on the guild you want the bot to join.\n"
"  %s"
msgstr "Per far entrare il bot in una gilda, incolla questo link nel tuo browser. \n"
"Nota: dovresti essere connesso al tuo account principale e avere i permessi \n"
"per gestire i server nella gilda che vuoi entrare nel bot.\n"
"  %s"

#: musicbot/bot.py:2370
#, python-format
msgid "Got None for bound channel with ID:  %d"
msgstr "Non c'è nessuno per il canale collegato con ID:  %d"

#: musicbot/bot.py:2376
#, python-format
msgid "Cannot bind to non Messageable channel with ID:  %d"
msgstr "Impossibile collegare al canale non Messaggiabile con ID:  %d"

#: musicbot/bot.py:2390
msgid "Bound to text channels:"
msgstr "Legato ai canali di testo:"

#: musicbot/bot.py:2404 musicbot/bot.py:2454
#, python-format
msgid " - %(guild)s/%(channel)s"
msgstr " - %(guild)s/%(channel)s"

#: musicbot/bot.py:2408 musicbot/bot.py:2410
msgid "Not bound to any text channels"
msgstr "Non associato a nessun canale di testo"

#: musicbot/bot.py:2421
#, python-format
msgid "Got None for auto join channel with ID:  %d"
msgstr "Non c'è nessuno per entrare automaticamente nel canale con ID:  %d"

#: musicbot/bot.py:2427
#, python-format
msgid "Cannot auto join a Private/Non-Guild channel with ID:  %d"
msgstr "Impossibile entrare automaticamente in un canale Privato/Non-Gilda con ID:  %d"

#: musicbot/bot.py:2435
#, python-format
msgid "Cannot auto join to non-connectable channel with ID:  %d"
msgstr "Impossibile entrare automaticamente nel canale non connettibile con ID:  %d"

#: musicbot/bot.py:2451
msgid "Auto joining voice channels:"
msgstr "Collegamento automatico canali vocali:"

#: musicbot/bot.py:2459 musicbot/bot.py:2462
msgid "Not auto joining any voice channels"
msgstr "Non si unisce automaticamente a nessun canale vocale"

#: musicbot/bot.py:2475
#, python-format
msgid "Detected missing config options!\n\n"
"Problem:\n"
"  You config options file is missing some options.\n"
"  Default settings will be used for these options.\n"
"  Here is a list of options we didn't find:\n"
"  %(missing)s\n\n"
"Solution:\n"
"  Copy new options from the example options file.\n"
"  Or use the config command to set and save them.\n\n"
msgstr "Rilevato le opzioni di configurazione mancanti!\n\n"
"Problema:\n"
"  Il file delle opzioni di configurazione manca di alcune opzioni.\n"
"  Le impostazioni predefinite saranno utilizzate per queste opzioni.\n"
"  Ecco un elenco di opzioni che non abbiamo trovato:\n"
"  %(missing)s\n\n"
"Soluzione:\n"
"  Copia nuove opzioni dal file delle opzioni di esempio.\n"
"  Oppure usa il comando di configurazione per impostarli e salvarli.\n\n"

#: musicbot/bot.py:2504
#, python-format
msgid "Event on_ready has fired %s times"
msgstr "L'evento on_ready ha sparato %s volte"

#: musicbot/bot.py:2520
msgid "Getting application info."
msgstr "Ottenere informazioni sull'applicazione."

#: musicbot/bot.py:2538
msgid "Ensuring data folders exist"
msgstr "Assicurare l'esistenza di cartelle dati"

#: musicbot/bot.py:2553
msgid "Validating config"
msgstr "Convalida configurazione"

#: musicbot/bot.py:2556
msgid "Validating permissions config"
msgstr "Convalida configurazione permessi"

#: musicbot/bot.py:2566
msgid "Disabled"
msgstr "Disabilitato"

#: musicbot/bot.py:2566
msgid "Enabled"
msgstr "Abilitato"

#: musicbot/bot.py:2569
msgid "Options:"
msgstr "Opzione:"

#: musicbot/bot.py:2571
#, python-format
msgid "  Command prefix: %s"
msgstr "  Prefisso dei comandi: %s"

#: musicbot/bot.py:2572
#, python-format
msgid "  Default volume: %d%%"
msgstr "  Volume predefinito: %d%%"

#: musicbot/bot.py:2574
#, python-format
msgid "  Skip threshold: %(num)d votes or %(percent).0f%%"
msgstr "  Salta soglia: %(num)d voti o %(percent).0f%%"

#: musicbot/bot.py:2581
#, python-format
msgid "  Now Playing @mentions: %s"
msgstr "  Riproduzione Di @mentions: %s"

#: musicbot/bot.py:2584
#, python-format
msgid "  Auto-Summon: %s"
msgstr "  Auto-Evocazione: %s"

#: musicbot/bot.py:2586
#, python-format
msgid "  Auto-Playlist: %(status)s (order: %(order)s)"
msgstr "  Scaletta Automatica: %(status)s (ordine: %(order)s)"

#: musicbot/bot.py:2589
msgid "random"
msgstr "casuale"

#: musicbot/bot.py:2589
msgid "sequential"
msgstr "sequenziale"

#: musicbot/bot.py:2594
#, python-format
msgid "  Auto-Pause: %s"
msgstr "  Auto-Pausa: %s"

#: musicbot/bot.py:2596
#, python-format
msgid "  Delete Messages: %s"
msgstr "  Elimina Messaggi: %s"

#: musicbot/bot.py:2601
#, python-format
msgid "    Delete Invoking: %s"
msgstr "    Elimina Invoking: %s"

#: musicbot/bot.py:2605
#, python-format
msgid "    Delete Now Playing: %s"
msgstr "    Elimina In Riproduzione: %s"

#: musicbot/bot.py:2608
#, python-format
msgid "  Debug Mode: %s"
msgstr "  Modalità Debug: %s"

#: musicbot/bot.py:2610
#, python-format
msgid "  Downloaded songs will be %s"
msgstr "  I brani scaricati saranno %s"

#: musicbot/bot.py:2614
#, python-format
msgid "    Delete if unused for %d days"
msgstr "    Elimina se non utilizzato per %d giorni"

#: musicbot/bot.py:2617
#, python-format
msgid "    Delete if size exceeds %s"
msgstr "    Elimina se la dimensione supera %s"

#: musicbot/bot.py:2620
#, python-format
msgid "  Status message: %s"
msgstr "  Messaggio di stato: %s"

#: musicbot/bot.py:2622
#, python-format
msgid "  Write current songs to file: %s"
msgstr "  Scrivi i brani attuali sul file: %s"

#: musicbot/bot.py:2626
#, python-format
msgid "  Author insta-skip: %s"
msgstr "  Autore insta-skip: %s"

#: musicbot/bot.py:2629
#, python-format
msgid "  Embeds: %s"
msgstr "  Embeds: %s"

#: musicbot/bot.py:2631
#, python-format
msgid "  Spotify integration: %s"
msgstr "  Integrazione Spotify: %s"

#: musicbot/bot.py:2634
#, python-format
msgid "  Legacy skip: %s"
msgstr "  Salto legacy: %s"

#: musicbot/bot.py:2636
#, python-format
msgid "  Leave non owners: %s"
msgstr "  Lascia non proprietari: %s"

#: musicbot/bot.py:2640
#, python-format
msgid "  Leave inactive VC: %s"
msgstr "  Lascia Vc inattivo: %s"

#: musicbot/bot.py:2645
#, python-format
msgid "    Timeout: %s seconds"
msgstr "    Timeout: %s secondi"

#: musicbot/bot.py:2649
#, python-format
msgid "  Leave at song end/empty queue: %s"
msgstr "  Abbandona alla coda fine/vuota: %s"

#: musicbot/bot.py:2653
#, python-format
msgid "  Leave when player idles: %s"
msgstr "  Abbandona quando il giocatore è inattivo: %s"

#: musicbot/bot.py:2657
#, python-format
msgid "    Timeout: %d seconds"
msgstr "    Timeout: %d secondi"

#: musicbot/bot.py:2658
#, python-format
msgid "  Self Deafen: %s"
msgstr "  Self Deafen: %s"

#: musicbot/bot.py:2660
#, python-format
msgid "  Per-server command prefix: %s"
msgstr "  Prefisso di comando per server: %s"

#: musicbot/bot.py:2663
#, python-format
msgid "  Search List: %s"
msgstr "  Elenco Di Ricerca: %s"

#: musicbot/bot.py:2665
#, python-format
msgid "  Round Robin Queue: %s"
msgstr "  Coda Round Robin: %s"

#: musicbot/bot.py:2695
#, python-format
msgid "The requested song `%(subject)s` is blocked by the song block list."
msgstr "La canzone richiesta `%(subject)s` è bloccata dalla lista dei brani."

#: musicbot/bot.py:2706
msgid "Attempted to handle Voice Channel inactivity, but Bot is not in voice..."
msgstr "Tentativo di gestire l'inattività del canale vocale, ma Bot non è in voce..."

#: musicbot/bot.py:2713
#, python-format
msgid "Channel activity already waiting in guild: %s"
msgstr "Attività canale già in attesa nella gilda: %s"

#: musicbot/bot.py:2723
#, python-format
msgid "Channel activity waiting %(time)d seconds to leave channel: %(channel)s"
msgstr "Attività del canale in attesa di %(time)d secondi per lasciare il canale: %(channel)s"

#: musicbot/bot.py:2738
#, python-format
msgid "Channel activity timer for %s has expired. Disconnecting."
msgstr "Il timer di attività del canale per %s è scaduto. Disconnessione."

#: musicbot/bot.py:2744
#, python-format
msgid "Channel activity timer canceled for: %(channel)s in %(guild)s"
msgstr "Timer attività canale annullato per: %(channel)s in %(guild)s"

#: musicbot/bot.py:2772
#, python-format
msgid "Ignoring player inactivity in auto-joined channel:  %s"
msgstr "Ignorare l'inattività del giocatore nel canale auto-entrato:  %s"

#: musicbot/bot.py:2779
#, python-format
msgid "Player activity timer already waiting in guild: %s"
msgstr "Timer attività giocatore già in attesa nella gilda: %s"

#: musicbot/bot.py:2787
#, python-format
msgid "Player activity timer waiting %(time)d seconds to leave channel: %(channel)s"
msgstr "Timer attività giocatore in attesa di %(time)d secondi per lasciare il canale: %(channel)s"

#: musicbot/bot.py:2799
#, python-format
msgid "Player activity timer for %s has expired. Disconnecting."
msgstr "Il timer di attività del giocatore per %s è scaduto. Disconnessione."

#: musicbot/bot.py:2805 musicbot/bot.py:2810
#, python-format
msgid "Player activity timer canceled for: %(channel)s in %(guild)s"
msgstr "Timer attività giocatore annullato per: %(channel)s in %(guild)s"

#: musicbot/bot.py:2827
msgid "Player activity timer is being reset."
msgstr "Il timer dell'attività del giocatore è in fase di ripristino."

#: musicbot/bot.py:2940
msgid "No such command"
msgstr "Comando inesistente"

#: musicbot/bot.py:3016
msgid "You must mention a user or provide their ID number."
msgstr "È necessario menzionare un utente o fornire il loro numero di identità."

#: musicbot/bot.py:3021
msgid "Invalid sub-command given. Use `help blockuser` for usage examples."
msgstr "Sottocomando fornito non valido. Usa `help blockuser` per esempi di utilizzo."

#: musicbot/bot.py:3032
msgid "MusicBot could not find the user(s) you specified."
msgstr "MusicBot non ha trovato gli utenti specificati."

#: musicbot/bot.py:3039
msgid "The owner cannot be added to the block list."
msgstr "Il proprietario non può essere aggiunto all'elenco dei blocchi."

#: musicbot/bot.py:3043
#, python-format
msgid "Not adding user to block list, already blocked:  %(id)s/%(name)s"
msgstr "Non aggiungere l'utente alla lista di blocchi, già bloccato:  %(id)s/%(name)s"

#: musicbot/bot.py:3052
#, python-format
msgid "Not removing user from block list, not listed:  %(id)s/%(name)s"
msgstr "Non rimuovere l'utente dalla lista dei blocchi, non elencato:  %(id)s/%(name)s"

#: musicbot/bot.py:3069
msgid "Cannot add the users you listed, they are already added."
msgstr "Non è possibile aggiungere gli utenti elencati, sono già aggiunti."

#: musicbot/bot.py:3146
msgid "You must provide a song subject if no song is currently playing."
msgstr "Devi fornire un oggetto del brano se nessuna canzone è attualmente in riproduzione."

#: musicbot/bot.py:3152
msgid "Invalid sub-command given. Use `help blocksong` for usage examples."
msgstr "Sottocomando fornito non valido. Usa `help blocksong` per esempi di utilizzo."

#: musicbot/bot.py:3164
#, python-format
msgid "Subject `%(subject)s` is already in the song block list."
msgstr "L'oggetto `%(subject)s` è già nella lista dei brani."

#: musicbot/bot.py:3196
msgid "The subject is not in the song block list and cannot be removed."
msgstr "L'oggetto non è nell'elenco dei brani e non può essere rimosso."

#: musicbot/bot.py:3249
msgid "Invalid sub-command given. Use `help autoplaylist` for usage examples."
msgstr "Sottocomando fornito non valido. Usa `help autoplaylist` per esempi di utilizzo."

#: musicbot/bot.py:3256
msgid "The supplied song link is invalid"
msgstr "Il link del brano fornito non è valido"

#: musicbot/bot.py:3262
msgid "The queue is empty. Add some songs with a play command!"
msgstr "La coda è vuota. Aggiungi alcuni brani con un comando di riproduzione!"

#: musicbot/bot.py:3290
msgid "This song is already in the autoplaylist."
msgstr "Questo brano è già nella playlist automatica."

#: musicbot/bot.py:3307
msgid "This song is not yet in the autoplaylist."
msgstr "Questa canzone non è ancora nella playlist automatica."

#: musicbot/bot.py:3337
msgid "You must provide a playlist filename."
msgstr "È necessario fornire un nome file playlist."

#: musicbot/bot.py:3428
msgid "You are not allowed to request playlists"
msgstr "Non sei autorizzato a richiedere playlist"

#: musicbot/bot.py:3436
#, python-format
msgid "Playlist has too many entries (%(songs)s but max is %(max)s)"
msgstr "La playlist ha troppe voci (%(songs)s ma il massimo è %(max)s)"

#: musicbot/bot.py:3447
#, python-format
msgid "The playlist entries will exceed your queue limit.\n"
"There are %(songs)s in the list, and %(queued)s already in queue.\n"
"The limit is %(max)s for your group."
msgstr "Le voci della playlist supereranno il limite della tua coda.\n"
"Ci sono %(songs)s nella lista, e %(queued)s già in coda.\n"
"Il limite è %(max)s per il tuo gruppo."

#: musicbot/bot.py:3471
msgid "Ignoring auto-pause due to network outage."
msgstr "Ignorare la pausa automatica a causa dell'interruzione della rete."

#: musicbot/bot.py:3476
msgid "MusicPlayer has no VoiceClient or has no channel data, cannot process auto-pause."
msgstr "MusicPlayer non ha VoiceClient o non ha dati di canale, non è possibile elaborare la pausa automatica."

#: musicbot/bot.py:3487
msgid "Already processing auto-pause, ignoring this event."
msgstr "Elaborazione della pausa automatica, ignorando questo evento."

#: musicbot/bot.py:3495
#, python-format
msgid "%sVoiceClient not connected, waiting %s seconds to handle auto-pause in guild:  %s"
msgstr "%sVoiceClient non connesso, in attesa di %s secondi per gestire la pausa automatica nella gilda:  %s"

#: musicbot/bot.py:3503
msgid "Auto-pause waiting was cancelled."
msgstr "Auto-pausa attesa è stata annullata."

#: musicbot/bot.py:3510
msgid "A new MusicPlayer is being connected, ignoring old auto-pause event."
msgstr "Un nuovo MusicPlayer è in fase di connessione, ignorando il vecchio evento di pausa automatica."

#: musicbot/bot.py:3526
#, python-format
msgid "Playing in an empty voice channel, running auto pause for guild: %s"
msgstr "Riproduzione in un canale vocale vuoto, in pausa automatica per gilda: %s"

#: musicbot/bot.py:3533
#, python-format
msgid "Previously auto paused player is unpausing for guild: %s"
msgstr "Il giocatore precedentemente in pausa automatica è in pausa per la gilda: %s"

#: musicbot/bot.py:3764
msgid "Cannot use seek if there is nothing playing."
msgstr "Impossibile usare la ricerca se non c'è niente di giocare."

#: musicbot/bot.py:3769
msgid "Cannot use seek on current track, it has an unknown duration."
msgstr "Impossibile usare la ricerca sulla traccia corrente, ha una durata sconosciuta."

#: musicbot/bot.py:3775
msgid "Seeking is not supported for streams."
msgstr "La ricerca non è supportata per gli streaming."

#: musicbot/bot.py:3785
msgid "Cannot use seek without a time to position playback."
msgstr "Impossibile usare la ricerca senza un tempo per posizionare la riproduzione."

#: musicbot/bot.py:3803
#, python-format
msgid "Could not convert `%(input)s` to a valid time in seconds."
msgstr "Impossibile convertire `%(input)s` in un tempo valido in secondi."

#: musicbot/bot.py:3816
#, python-format
msgid "Cannot seek to `%(input)s` (`%(seconds)s` seconds) in the current track with a length of `%(progress)s / %(total)s`"
msgstr "Impossibile cercare `%(input)s` (`%(seconds)s` secondi) nella traccia corrente con una lunghezza di `%(progress)s / %(total)s`"

#: musicbot/bot.py:3877
msgid "Invalid sub-command. Use the command `help repeat` for usage examples."
msgstr "Sottocomando non valido. Usa il comando `help repeat` per gli esempi di utilizzo."

#: musicbot/bot.py:3915
msgid "The player is not currently looping."
msgstr "Il giocatore non si sta caricando."

#: musicbot/bot.py:3971
msgid "Song positions must be integers!"
msgstr "Le posizioni brano devono essere intere!"

#: musicbot/bot.py:3976
msgid "You gave a position outside the playlist size!"
msgstr "Hai dato una posizione fuori dalla dimensione della playlist!"

#: musicbot/bot.py:4025
msgid "Could not prompt for playlist playback, no message to add reactions to."
msgstr "Impossibile richiedere la riproduzione della playlist, nessun messaggio a cui aggiungere reazioni."

#: musicbot/bot.py:4114
msgid "Local media playback is not enabled."
msgstr "La riproduzione multimediale locale non è abilitata."

#: musicbot/bot.py:4155
msgid "Spotify URL is invalid or not currently supported."
msgstr "L'URL di Spotify non è valido o non è attualmente supportato."

#: musicbot/bot.py:4159
msgid "Detected a Spotify URL, but Spotify is not enabled."
msgstr "Rilevato un URL Spotify, ma Spotify non è abilitato."

#: musicbot/bot.py:4169 musicbot/bot.py:4374
#, python-format
msgid "You have reached your enqueued song limit (%(max)s)"
msgstr "Hai raggiunto il limite di canzone in coda (%(max)s)"

#: musicbot/bot.py:4175 musicbot/bot.py:4380 musicbot/bot.py:4474
msgid "Karaoke mode is enabled, please try again when its disabled!"
msgstr "La modalità Karaoke è abilitata, riprova quando è disabilitata!"

#: musicbot/bot.py:4187
msgid "Issue with extract_info(): "
msgstr "Problema con extract_info(): "

#: musicbot/bot.py:4191 musicbot/bot.py:4396
#, python-format
msgid "Failed to extract info due to error:\n"
"%(raw_error)s"
msgstr "Impossibile estrarre le informazioni a causa di errore:\n"
"%(raw_error)s"

#: musicbot/bot.py:4197
msgid "That video cannot be played. Try using the stream command."
msgstr "Questo video non può essere riprodotto. Prova a usare il comando streaming."

#: musicbot/bot.py:4208
#, python-format
msgid "YouTube search returned no results for:  %(url)s"
msgstr "La ricerca su YouTube non ha prodotto alcun risultato per:  %(url)s"

#: musicbot/bot.py:4238
#, python-format
msgid "Processed %(number)d of %(total)d songs in %(time).3f seconds at %(time_per).2f s/song"
msgstr "Elaborato %(number)d di %(total)d canzoni in %(time).3f secondi in %(time_per).2f s/song"

#: musicbot/bot.py:4249
#, python-format
msgid "No songs were added, all songs were over max duration (%(max)s seconds)"
msgstr "Nessuna canzone aggiunta, tutte le canzoni hanno una durata massima (%(max)s secondi)"

#: musicbot/bot.py:4265
msgid "Extracted an entry with 'youtube:playlist' as extractor key"
msgstr "Estratto una voce con 'youtube:playlist' come chiave estrattore"

#: musicbot/bot.py:4277
#, python-format
msgid "Song duration exceeds limit (%(length)s > %(max)s)"
msgstr "La durata del brano supera il limite (%(length)s > %(max)s)"

#: musicbot/bot.py:4295
#, python-format
msgid "Added song(s) at position %s"
msgstr "Aggiunti brani alla posizione %s"

#: musicbot/bot.py:4332
#, python-format
msgid "Cannot estimate time until playing for position: %d"
msgstr "Impossibile stimare il tempo di riproduzione per la posizione: %d"

#: musicbot/bot.py:4393
#, python-format
msgid "Failed to get info from the stream request: %s"
msgstr "Recupero delle informazioni dalla richiesta dello stream non riuscito: %s"

#: musicbot/bot.py:4402
msgid "Streaming playlists is not yet supported."
msgstr "Le playlist in streaming non sono ancora supportate."

#: musicbot/bot.py:4468
#, python-format
msgid "You have reached your playlist item limit (%(max)s)"
msgstr "Hai raggiunto il limite di elementi della playlist (%(max)s)"

#: musicbot/bot.py:4480
msgid "Please specify a search query.  Use `help search` for more information."
msgstr "Specifica una query di ricerca. Usa `help search` per maggiori informazioni."

#: musicbot/bot.py:4509
#, python-format
msgid "You cannot search for more than %(max)s videos"
msgstr "Non puoi cercare più di %(max)s video"

#: musicbot/bot.py:4834
#, python-format
msgid "Waiting for summon lock: %s"
msgstr "In attesa del blocco della voce: %s"

#: musicbot/bot.py:4837
#, python-format
msgid "Summon lock acquired for: %s"
msgstr "Evoca blocco acquisito per: %s"

#: musicbot/bot.py:4841
msgid "You are not connected to voice. Try joining a voice channel!"
msgstr "Non sei connesso alla voce. Prova ad unirti a un canale vocale!"

#: musicbot/bot.py:4866
#, python-format
msgid "Joining %(guild)s/%(channel)s"
msgstr "Entrare In %(guild)s/%(channel)s"

#: musicbot/bot.py:4928
msgid "MusicBot cannot follow a user that is not a member of the server."
msgstr "MusicBot non può seguire un utente che non è un membro del server."

#: musicbot/bot.py:4956
msgid "Player is not playing."
msgstr "Il giocatore non sta giocando."

#: musicbot/bot.py:5053
msgid "Nothing in the queue to remove!"
msgstr "Niente nella coda da rimuovere!"

#: musicbot/bot.py:5074
#, python-format
msgid "Nothing found in the queue from user `%(user)s`"
msgstr "Niente trovato nella coda dall'utente `%(user)s`"

#: musicbot/bot.py:5079 musicbot/bot.py:5114
msgid "You do not have the permission to remove that entry from the queue.\n"
"You must be the one who queued it or have instant skip permissions."
msgstr "Non hai il permesso di rimuovere questa voce dalla coda.\n"
"Devi essere colui che l'ha accodata o che ha i permessi di salto istantanei."

#: musicbot/bot.py:5090 musicbot/bot.py:5095
msgid "Invalid entry number. Use the queue command to find queue positions."
msgstr "Numero di voce non valido. Usa il comando coda per trovare le posizioni della coda."

#: musicbot/bot.py:5145
msgid "Can't skip! The player is not playing!"
msgstr "Impossibile saltare! Il giocatore non sta giocando!"

#: musicbot/bot.py:5197
msgid "You do not have permission to force skip a looped song."
msgstr "Non hai il permesso di forzare un brano in loop."

#: musicbot/bot.py:5216
msgid "You do not have permission to force skip."
msgstr "Non hai il permesso di forzare il salto."

#: musicbot/bot.py:5253 musicbot/bot.py:5286
msgid "You do not have permission to skip a looped song."
msgstr "Non hai il permesso di saltare una canzone in loop."

#: musicbot/bot.py:5336
#, python-format
msgid "`%(new_volume)s` is not a valid number"
msgstr "`%(new_volume)s` non è un numero valido"

#: musicbot/bot.py:5357
#, python-format
msgid "Unreasonable volume change provided: %(old_volume)s%(adjustment)s is %(new_volume)s.\n"
"Volume can only be set from 1 to 100."
msgstr "Variazione di volume irragionevole fornita: %(old_volume)s%(adjustment)s è %(new_volume)s.\n"
"Il volume può essere impostato solo da 1 a 100."

#: musicbot/bot.py:5367
#, python-format
msgid "Unreasonable volume provided: %(volume)s. Provide a value between 1 and 100."
msgstr "Volume irragionevole fornito: %(volume)s. Fornire un valore compreso tra 1 e 100."

#: musicbot/bot.py:5391
msgid "No track is playing, cannot set speed.\n"
"Use the config command to set a default playback speed."
msgstr "Nessuna traccia in riproduzione, non è possibile impostare la velocità.\n"
"Utilizzare il comando di configurazione per impostare una velocità di riproduzione predefinita."

#: musicbot/bot.py:5399
msgid "Speed cannot be applied to streamed media."
msgstr "La velocità non può essere applicata ai media in streaming."

#: musicbot/bot.py:5404
msgid "You must provide a speed to set."
msgstr "Devi fornire una velocità da impostare."

#: musicbot/bot.py:5413
msgid "The speed you provided is invalid. Use a number between 0.5 and 100."
msgstr "La velocità fornita non è valida. Usa un numero compreso tra 0.5 e 100."

#: musicbot/bot.py:5471 musicbot/bot.py:5587 musicbot/bot.py:6519
#, python-format
msgid "Invalid option for command: `%(option)s`"
msgstr "Opzione non valida per il comando: `%(option)s`"

#: musicbot/bot.py:5485
#, python-format
msgid "Failed to save aliases due to error:\n"
"`%(raw_error)s`"
msgstr "Impossibile salvare gli alias a causa di errore:\n"
"`%(raw_error)s`"

#: musicbot/bot.py:5492
msgid "You must supply an alias and a command to alias"
msgstr "Devi fornire un alias e un comando all'alias"

#: musicbot/bot.py:5507
msgid "You must supply an alias name to remove."
msgstr "Devi fornire un nome alias da rimuovere."

#: musicbot/bot.py:5512
#, python-format
msgid "The alias `%(alias)s` does not exist."
msgstr "L'alias `%(alias)s` non esiste."

#: musicbot/bot.py:5570
msgid "Config cannot use channel and user mentions at the same time."
msgstr "La configurazione non può usare le menzioni del canale e dell'utente allo stesso tempo."

#: musicbot/bot.py:5673
#, python-format
msgid "Unable to reload Config due to the following error:\n"
"%(raw_error)s"
msgstr "Impossibile ricaricare la configurazione a causa del seguente errore:\n"
"%(raw_error)s"

#: musicbot/bot.py:5691
msgid "Could not resolve section name from option name. Please provide a valid section and option name."
msgstr "Impossibile risolvere il nome della sezione dal nome dell'opzione. Si prega di fornire una sezione e un nome di opzione validi."

#: musicbot/bot.py:5695
msgid "The option given is ambiguous, please provide a section name."
msgstr "L'opzione data è ambigua, si prega di fornire un nome di sezione."

#: musicbot/bot.py:5701
msgid "You must provide a section name and option name for this command."
msgstr "Devi fornire un nome di sezione e un nome di opzione per questo comando."

#: musicbot/bot.py:5717
#, python-format
msgid "The section `%(section)s` is not available.\n"
"The available sections are:  %(sections)s"
msgstr "La sezione `%(section)s` non è disponibile.\n"
"Le sezioni disponibili sono:  %(sections)s"

#: musicbot/bot.py:5725
#, python-format
msgid "The option `%(option)s` is not available."
msgstr "L'opzione `%(option)s` non è disponibile."

#: musicbot/bot.py:5753
#, python-format
msgid "Option `%(option)s` is not editable. Cannot save to disk."
msgstr "L'opzione `%(option)s` non è modificabile. Impossibile salvare sul disco."

#: musicbot/bot.py:5762
#, python-format
msgid "Failed to save the option:  `%(option)s`"
msgstr "Impossibile salvare l'opzione: `%(option)s`"

#: musicbot/bot.py:5777
#, python-format
msgid "Option `%(option)s` is not editable, value cannot be displayed."
msgstr "L'opzione `%(option)s` non è modificabile, il valore non può essere visualizzato."

#: musicbot/bot.py:5800
#, python-format
msgid "Option `%(option)s` is not editable. Cannot update setting."
msgstr "L'opzione `%(option)s` non è modificabile. Impossibile aggiornare l'impostazione."

#: musicbot/bot.py:5806 musicbot/bot.py:6732
msgid "You must provide a section, option, and value for this sub command."
msgstr "Devi fornire una sezione, un'opzione e un valore per questo sottocomando."

#: musicbot/bot.py:5810
#, python-format
msgid "Doing set with on %(config)s == %(value)s"
msgstr "Impostazione con %(config)s == %(value)s"

#: musicbot/bot.py:5817
#, python-format
msgid "Option `%(option)s` was not updated!"
msgstr "L'opzione `%(option)s` non è stata aggiornata!"

#: musicbot/bot.py:5833
#, python-format
msgid "Option `%(option)s` is not editable. Cannot reset to default."
msgstr "L'opzione `%(option)s` non è modificabile. Impossibile ripristinare il valore predefinito."

#: musicbot/bot.py:5845
#, python-format
msgid "Resetting %(config)s to default %(value)s"
msgstr "Ripristino %(config)s a predefinito %(value)s"

#: musicbot/bot.py:5852
#, python-format
msgid "Option `%(option)s` was not reset to default!"
msgstr "L'opzione `%(option)s` non è stata ripristinata al predefinito!"

#: musicbot/bot.py:5885
msgid "The option command is deprecated, use the config command instead."
msgstr "Il comando opzione è deprecato, utilizza invece il comando di configurazione."

#: musicbot/bot.py:5905
msgid "Invalid option specified, use: info, update, or clear"
msgstr "Opzione specificata non valida, usa: info, aggiornamento o cancella"

#: musicbot/bot.py:5962
msgid "**Failed** to delete cache, check logs for more info..."
msgstr "**Non riuscito** per eliminare la cache, controlla i log per ulteriori informazioni..."

#: musicbot/bot.py:5997
msgid "Queue page argument must be a whole number."
msgstr "L'argomento pagina coda deve essere un numero intero."

#: musicbot/bot.py:6004
msgid "There are no songs queued! Queue something with a play command."
msgstr "Non ci sono brani in coda! Coda qualcosa con un comando di riproduzione."

#: musicbot/bot.py:6013
#, python-format
msgid "Requested page number is out of bounds.\n"
"There are **%(total)s** pages."
msgstr "Il numero di pagina richiesto è fuori dai limiti.\n"
"Ci sono ** pagine%(total)s**."

#: musicbot/bot.py:6056
msgid "Skipped the current playlist entry."
msgstr "Ha saltato la voce attuale della scaletta."

#: musicbot/bot.py:6099
msgid "Not enough entries to paginate the queue."
msgstr "Non ci sono abbastanza voci per paginare la coda."

#: musicbot/bot.py:6103
msgid "Could not post queue message, no message to add reactions to."
msgstr "Impossibile postare il messaggio in coda, nessun messaggio a cui aggiungere reazioni."

#: musicbot/bot.py:6105
msgid "Try that again. MusicBot couldn't make or get a reference to the queue message.\n"
"If the issue persists, file a bug report."
msgstr "Riprova. MusicBot non ha potuto fare o ottenere un riferimento al messaggio di coda.\n"
"Se il problema persiste, file una segnalazione di bug."

#: musicbot/bot.py:6215
msgid "Cannot use purge on private DM channel."
msgstr "Impossibile usare lo spurgo sul canale DM privato."

#: musicbot/bot.py:6247
msgid "The given URL was not a valid URL."
msgstr "L'URL fornito non era un URL valido."

#: musicbot/bot.py:6257
#, python-format
msgid "Could not extract info from input url\n"
"%(raw_error)s\n"
msgstr "Impossibile estrarre le informazioni dall'url di input\n"
"%(raw_error)s\n"

#: musicbot/bot.py:6262
msgid "This does not seem to be a playlist."
msgstr "Questa non sembra essere una playlist."

#: musicbot/bot.py:6425
msgid "Invalid user ID or server nickname, please double-check the ID and try again."
msgstr "ID utente o soprannome del server non validi, si prega di ricontrollare l'ID e riprovare."

#: musicbot/bot.py:6432
msgid "Could not determine the discord User.  Try again."
msgstr "Impossibile determinare l'utente discord. Riprova."

#: musicbot/bot.py:6503
msgid "Permissions cannot use channel and user mentions at the same time."
msgstr "Le autorizzazioni non possono usare le menzioni del canale e dell'utente allo stesso tempo."

#: musicbot/bot.py:6538
#, python-format
msgid "Unable to reload Permissions due to an error:\n"
"%(raw_error)s"
msgstr "Impossibile ricaricare i permessi a causa di un errore:\n"
"%(raw_error)s"

#: musicbot/bot.py:6575
msgid "You must provide a group or option name for this command."
msgstr "Devi fornire un nome di gruppo o di opzione per questo comando."

#: musicbot/bot.py:6579
msgid "You must provide a group, option, and value to set for this command."
msgstr "Devi fornire un gruppo, un'opzione e un valore da impostare per questo comando."

#: musicbot/bot.py:6593
#, python-format
msgid "The %(option)s sub-command requires a group and permission name."
msgstr "Il sottocomando %(option)s richiede un nome di gruppo e di permesso."

#: musicbot/bot.py:6605
#, python-format
msgid "The group `%(group)s` is not available.\n"
"The available groups are:  %(sections)s"
msgstr "Il gruppo `%(group)s` non è disponibile.\n"
"I gruppi disponibili sono:  %(sections)s"

#: musicbot/bot.py:6616
#, python-format
msgid "The permission `%(option)s` is not available."
msgstr "Il permesso `%(option)s` non è disponibile."

#: musicbot/bot.py:6652
#, python-format
msgid "Cannot add group `%(group)s` it already exists."
msgstr "Impossibile aggiungere il gruppo `%(group)s` che esiste già."

#: musicbot/bot.py:6670
msgid "Cannot remove built-in group."
msgstr "Impossibile rimuovere il gruppo integrato."

#: musicbot/bot.py:6688 musicbot/bot.py:6727
msgid "The owner group is not editable."
msgstr "Il gruppo proprietario non è modificabile."

#: musicbot/bot.py:6696
#, python-format
msgid "Failed to save the group:  `%(group)s`"
msgstr "Impossibile salvare il gruppo: `%(group)s`"

#: musicbot/bot.py:6736
#, python-format
msgid "Doing set on %(option)s with value: %(value)s"
msgstr "Impostazione %(option)s con valore: %(value)s"

#: musicbot/bot.py:6743
#, python-format
msgid "Permission `%(option)s` was not updated!"
msgstr "Il permesso `%(option)s` non è stato aggiornato!"

#: musicbot/bot.py:6784
msgid "Failed to change username. Did you change names too many times?\n"
"Remember name changes are limited to twice per hour.\n"
msgstr "Impossibile modificare il nome utente. Hai cambiato i nomi troppe volte?\n"
"Ricorda che i cambiamenti di nome sono limitati a due volte all'ora.\n"

#: musicbot/bot.py:6790
#, python-format
msgid "Failed to change username due to error:  \n"
"%(raw_error)s"
msgstr "Impossibile modificare il nome utente a causa di errore:  \n"
"%(raw_error)s"

#: musicbot/bot.py:6812
msgid "Unable to change nickname: no permission."
msgstr "Impossibile cambiare il soprannome: nessun permesso."

#: musicbot/bot.py:6820
#, python-format
msgid "Failed to set nickname due to error:  \n"
"%(raw_error)s"
msgstr "Impossibile impostare il soprannome a causa di errore:  \n"
"%(raw_error)s"

#: musicbot/bot.py:6860
msgid "Custom emoji must be from this server to use as a prefix."
msgstr "Le emoji personalizzate devono provenire da questo server per essere utilizzate come prefisso."

#: musicbot/bot.py:6877
msgid "Prefix per server is not enabled!\n"
"Use the config command to update the prefix instead."
msgstr "Prefisso per server non è abilitato!\n"
"Usa il comando di configurazione per aggiornare il prefisso."

#: musicbot/bot.py:6905
msgid "You must provide a URL or attach a file."
msgstr "È necessario fornire un URL o allegare un file."

#: musicbot/bot.py:6915
#, python-format
msgid "Unable to change avatar due to error:  \n"
"%(raw_error)s"
msgstr "Impossibile cambiare l'avatar a causa dell'errore:  \n"
"%(raw_error)s"

#: musicbot/bot.py:6940
#, python-format
msgid "MusicBot found a %s with no guild!  This could be a problem."
msgstr "MusicBot ha trovato un %s senza gilda! Questo potrebbe essere un problema."

#: musicbot/bot.py:6952
#, python-format
msgid "Not currently connected to server `%(guild)s`"
msgstr "Attualmente non connesso al server `%(guild)s`"

#: musicbot/bot.py:6993
msgid "Invalid option given, use one of:  soft, full, upgrade, uppip, or upgit"
msgstr "Opzione fornita non valida, usa una di: soft, full, upgrade, uppip, or upgit"

#: musicbot/bot.py:7102
msgid "You must provide an ID or name."
msgstr "Devi fornire un ID o un nome."

#: musicbot/bot.py:7113
#, python-format
msgid "No guild was found with the ID or name `%(input)s`"
msgstr "Nessuna gilda trovata con l'ID o il nome `%(input)s`"

#: musicbot/bot.py:7172
#, python-format
msgid "Activating debug breakpoint ID: %(uuid)s"
msgstr "Attivazione ID breakpoint di debug: %(uuid)s"

#: musicbot/bot.py:7212
msgid "Could not import `objgraph`, is it installed?"
msgstr "Impossibile importare `objgraph`, è installato?"

#: musicbot/bot.py:7287
msgid "Debug code ran with eval()."
msgstr "Codice di debug eseguito con eval()."

#: musicbot/bot.py:7297
msgid "Debug code ran with exec()."
msgstr "Il codice di debug è funzionato con exec()."

#: musicbot/bot.py:7300
msgid "Debug code failed to execute."
msgstr "Impossibile eseguire il codice di debug."

#: musicbot/bot.py:7302
#, python-format
msgid "Failed to execute debug code:\n"
"%(py_code)s\n"
"Exception: ```\n"
"%(ex_name)s:\n"
"%(ex_text)s```"
msgstr "Esecuzione del codice di debug non riuscita:\n"
"%(py_code)s\n"
"Eccezione: ```\n"
"%(ex_name)s:\n"
"%(ex_text)s```"

#: musicbot/bot.py:7337 musicbot/bot.py:7398
#, python-format
msgid "Sub-command must be one of: %(options)s"
msgstr "Il sottocomando deve essere uno di: %(options)s"

#: musicbot/bot.py:7439
msgid "Could not locate git executable."
msgstr "Impossibile individuare git eseguibile."

#: musicbot/bot.py:7483
msgid "Failed while checking for updates via git command."
msgstr "Impossibile controllare gli aggiornamenti tramite il comando git."

#: musicbot/bot.py:7512
msgid "Package missing meta in pip report."
msgstr "Pacchetto meta mancante nel rapporto pip."

#: musicbot/bot.py:7526
msgid "Failed to get pip update status due to some error."
msgstr "Impossibile ottenere lo stato di aggiornamento dei pip a causa di qualche errore."

#: musicbot/bot.py:7586
msgid "Got a strange voice client entry."
msgstr "Ottenuto una strana voce client vocale."

#: musicbot/bot.py:7688
msgid "Cookies already enabled."
msgstr "Cookie già abilitati."

#: musicbot/bot.py:7695
msgid "Cookies must be uploaded to be enabled. (Missing cookies file.)"
msgstr "I cookie devono essere caricati per essere abilitati. (File dei cookie)."

#: musicbot/bot.py:7708
#, python-format
msgid "Could not enable cookies due to error:  %(raw_error)s"
msgstr "Impossibile abilitare i cookie a causa di errore:  %(raw_error)s"

#: musicbot/bot.py:7722
#, python-format
msgid "Could not rename cookies file due to error:  %(raw_error)s\n"
"Cookies temporarily disabled and will be re-enabled on next restart."
msgstr "Impossibile rinominare il file cookie a causa di errore:  %(raw_error)s\n"
"I cookie sono temporaneamente disabilitati e saranno riattivati al prossimo riavvio."

#: musicbot/bot.py:7731
msgid "No attached uploads were found, try again while uploading a cookie file."
msgstr "Non sono stati trovati caricamenti allegati, riprova durante il caricamento di un file di cookie."

#: musicbot/bot.py:7740
#, python-format
msgid "Could not remove old, disabled cookies file:  %(raw_error)s"
msgstr "Impossibile rimuovere il vecchio file cookie disabilitato:  %(raw_error)s"

#: musicbot/bot.py:7749
#, python-format
msgid "Error downloading the cookies file from discord:  %(raw_error)s"
msgstr "Errore nel scaricare il file dei cookie da discord:  %(raw_error)s"

#: musicbot/bot.py:7754
#, python-format
msgid "Could not save cookies to disk:  %(raw_error)s"
msgstr "Impossibile salvare i cookie sul disco:  %(raw_error)s"

#: musicbot/bot.py:7772
#, python-format
msgid "Got a message with no channel, somehow:  %s"
msgstr "Hai un messaggio senza canale, in qualche modo:  %s"

#: musicbot/bot.py:7800
#, python-format
msgid "Ignoring command from myself (%s)"
msgstr "Ignorare il comando da me stesso (%s)"

#: musicbot/bot.py:7808
#, python-format
msgid "Ignoring command from other bot (%s)"
msgstr "Ignora il comando da un altro bot (%s)"

#: musicbot/bot.py:7816
#, python-format
msgid "Ignoring command from channel of type:  %s"
msgstr "Ignora il comando dal canale di tipo:  %s"

#: musicbot/bot.py:7900
#, python-format
msgid "User in block list: %(id)s/%(name)s  tried command: %(command)s"
msgstr "Utente nella lista dei blocchi: %(id)s/%(name)s  comando provato: %(command)s"

#: musicbot/bot.py:7907
#, python-format
msgid "Message from %(id)s/%(name)s: %(message)s"
msgstr "Messaggio da %(id)s/%(name)s: %(message)s"

#: musicbot/bot.py:7941
#, python-format
msgid "This command is not allowed for your permissions group:  %(group)s"
msgstr "Questo comando non è consentito per il tuo gruppo di permessi:  %(group)s"

#: musicbot/bot.py:7976
msgid "This command requires you to be in a Voice channel."
msgstr "Questo comando richiede di essere in un canale vocale."

#: musicbot/bot.py:8060
#, python-format
msgid "Invalid command usage, missing values for params: %(params)r"
msgstr "Utilizzo del comando non valido, valori mancanti per i parametri: %(params)r"

#: musicbot/bot.py:8102
#, python-format
msgid "Error in %(command)s: %(err_name)s: %(err_text)s"
msgstr "Errore in %(command)s: %(err_name)s: %(err_text)s"

#: musicbot/bot.py:8126
#, python-format
msgid "Exception while handling command: %(command)s"
msgstr "Eccezione durante la gestione del comando: %(command)s"

#: musicbot/bot.py:8158
#, python-format
msgid "Cannot generate help for missing command:  %s"
msgstr "Impossibile generare l'aiuto per il comando mancante:  %s"

#: musicbot/bot.py:8162
#, python-format
msgid "Missing help data for command:  %s"
msgstr "Dati di aiuto mancanti per il comando:  %s"

#: musicbot/bot.py:8279
#, python-format
msgid "Leaving voice channel %s in %s due to inactivity."
msgstr "Lasciare il canale vocale %s in %s a causa di inattività."

#: musicbot/bot.py:8288
msgid "MusicBot has become connected."
msgstr "MusicBot si è connesso."

#: musicbot/bot.py:8292
msgid "MusicBot has become disconnected."
msgstr "MusicBot è stato disconnesso."

#: musicbot/bot.py:8297
#, python-format
msgid "Got a Socket Event:  %s"
msgstr "Ottenuto un Evento Socket:  %s"

#: musicbot/bot.py:8313
msgid "VoiceState updated before on_ready finished"
msgstr "Stato vocale aggiornato prima del termine _pronto"

#: musicbot/bot.py:8326
#, python-format
msgid "Ignoring %s in %s as it is a bound voice channel."
msgstr "Ignorare %s in %s in quanto è un canale vocale collegato."

#: musicbot/bot.py:8335
#, python-format
msgid "%s has been detected as empty. Handling timeouts."
msgstr "%s è stato rilevato come timeout vuoto."

#: musicbot/bot.py:8346
#, python-format
msgid "A user joined %s, cancelling timer."
msgstr "Un utente si è unito %s, annullando il timer."

#: musicbot/bot.py:8359
#, python-format
msgid "The bot got moved and the voice channel %s is empty. Handling timeouts."
msgstr "Il bot è stato spostato e il canale vocale %s è vuoto. Timeout gestione."

#: musicbot/bot.py:8368
#, python-format
msgid "The bot got moved and the voice channel %s is not empty."
msgstr "Il bot è stato spostato e il canale vocale %s non è vuoto."

#: musicbot/bot.py:8402
#, python-format
msgid "No longer following user %s"
msgstr "Non più seguendo l'utente %s"

#: musicbot/bot.py:8424
#, python-format
msgid "Following user `%(user)s` to channel:  %(channel)s"
msgstr "Seguire l'utente `%(user)s` al canale:  %(channel)s"

#: musicbot/bot.py:8444
msgid "VoiceState disconnect before.channel is None."
msgstr "VoiceState disconnetti before.channel non è nessuno."

#: musicbot/bot.py:8471
#, python-format
msgid "Disconnected from voice by Discord API in: %(guild)s/%(channel)s (Code: %(code)s) [S:%(state)s]"
msgstr "Disconnesso dalla voce dall'API di Discord in: %(guild)s/%(channel)s (Code: %(code)s) [S:%(state)s]"

#: musicbot/bot.py:8493
#, python-format
msgid "Cannot use auto-join channel with type: %(type)s  in guild:  %(guild)s"
msgstr "Impossibile usare il canale di auto-join con il tipo: %(type)s  nella gilda:  %(guild)s"

#: musicbot/bot.py:8500
#, python-format
msgid "Cannot find the auto-joined channel, was it deleted?  Guild:  %s"
msgstr "Impossibile trovare il canale di accesso automatico, è stato eliminato? Gilda:  %s"

#: musicbot/bot.py:8506
#, python-format
msgid "Reconnecting to auto-joined guild on channel:  %s"
msgstr "Riconnessione alla gilda auto-entrata sul canale:  %s"

#: musicbot/bot.py:8519
#, python-format
msgid "Cannot auto join channel:  %s"
msgstr "Impossibile entrare automaticamente nel canale:  %s"

#: musicbot/bot.py:8535
#, python-format
msgid "Bot has been added to guild: %s"
msgstr "Il bot è stato aggiunto alla gilda: %s"

#: musicbot/bot.py:8547
#, python-format
msgid "Left guild '%s' due to bot owner not found."
msgstr "Gilda sinistra '%s' a causa del proprietario del bot non trovato."

#: musicbot/bot.py:8561
#, python-format
msgid "Creating data folder for guild %s"
msgstr "Creazione cartella dati per la gilda %s"

#: musicbot/bot.py:8569
#, python-format
msgid "Bot has been removed from guild: %s"
msgstr "Il bot è stato rimosso dalla gilda: %s"

#: musicbot/bot.py:8570
msgid "Updated guild list:"
msgstr "Lista gilda aggiornata:"

#: musicbot/bot.py:8585
#, python-format
msgid "Guild \"%s\" has become available."
msgstr "Gilda \"%s\" è diventato disponibile."

#: musicbot/bot.py:8591
#, python-format
msgid "Resuming player in \"%s\" due to availability."
msgstr "Riprendere il giocatore in \"%s\" a causa della disponibilità."

#: musicbot/bot.py:8608
#, python-format
msgid "Guild \"%s\" has become unavailable."
msgstr "Gilda \"%s\" non è disponibile."

#: musicbot/bot.py:8614
#, python-format
msgid "Pausing player in \"%s\" due to unavailability."
msgstr "Pausa giocatore in \"%s\" a causa di indisponibilità."

#: musicbot/bot.py:8630
#, python-format
msgid "Guild update for:  %s"
msgstr "Aggiornamento gilda per:  %s"

#: musicbot/bot.py:8636
#, python-format
msgid "Guild attribute %(attr)s is now: %(new)s  -- Was: %(old)s"
msgstr "L'attributo della Gilda %(attr)s è ora: %(new)s  -- Era %(old)s"

#: musicbot/bot.py:8668
#, python-format
msgid "Channel update for:  %(channel)s  --  %(changes)s"
msgstr "Aggiornamento canale per:  %(channel)s  --  %(changes)s"

#: musicbot/config.py:86
#, python-format
msgid "Creating %s"
msgstr "Creazione %s"

#: musicbot/config.py:107
#, python-format
msgid "Loading config from:  %s"
msgstr "Caricamento configurazione da:  %s"

#: musicbot/config.py:1083
#, python-format
msgid "Error while reading config.\n\n"
"Problem:\n"
"  One or more required config option sections are missing.\n"
"  The missing sections are:\n"
"  %(sections)s\n\n"
"Solution:\n"
"  Repair your config options file.\n"
"  Each [Section] must appear only once, with no other text on the same line.\n"
"  Each section must have at least one option.\n"
"  Use the example options as a template or copy it from the repository."
msgstr "Errore durante la lettura della configurazione.\n\n"
"Problema:\n"
"  Mancano una o più sezioni di opzioni di configurazione richieste.\n"
"  Le sezioni mancanti sono: Soluzione\n"
"  %(sections)s\n\n"
":\n"
"  Riparare il file delle opzioni di configurazione.\n"
"  Ogni [Section] deve apparire una sola volta, senza nessun altro testo sulla stessa riga.\n"
"  Ogni sezione deve avere almeno un'opzione.\n"
"  Utilizzare le opzioni di esempio come modello o copiarlo dal repository."

#: musicbot/config.py:1113
#, python-format
msgid "Cannot store more than %s log files. Option LogsMaxKept will be limited instead."
msgstr "Impossibile memorizzare più di file di log %s . L'opzione LogsMaxKept sarà limitata."

#: musicbot/config.py:1121
msgid "Config option LogsDateFormat is empty and this will break log file rotation. Using default instead."
msgstr "Opzione di configurazione LogsDateFormat è vuoto e questo interromperà la rotazione del file di registro. Usando invece predefinito."

#: musicbot/config.py:1132
msgid "Error while validating config options.\n\n"
"Problem:\n"
"  Config option AudioCachePath is not a directory.\n\n"
"Solution:\n"
"  Make sure the path you configured is a path to a folder / directory."
msgstr "Errore durante la convalida delle opzioni di configurazione.\n\n"
"Problema:\n"
"  Opzione di configurazione AudioCachePath non è una directory.\n\n"
"Soluzione:\n"
"  Assicurati che il percorso configurato sia un percorso per una cartella / directory."

#: musicbot/config.py:1149
msgid "An exception was thrown while validating AudioCachePath."
msgstr "È stata lanciata un'eccezione durante la convalida di AudioCachePath."

#: musicbot/config.py:1153
#, python-format
msgid "Error while validating config options.\n\n"
"Problem:\n"
"  AudioCachePath config option could not be set due to an error:\n"
"  %(raw_error)s\n\n"
"Solution:\n"
"  Double check the setting is a valid, accessible directory path."
msgstr "Errore durante la convalida delle opzioni di configurazione.\n\n"
"Problema: l'opzione di configurazione\n"
"  AudioCachePath non può essere impostata a causa di un errore:\n"
"  %(raw_error)s\n\n"
"Soluzione:\n"
"  Doppio controllo l'impostazione è valida, percorso directory accessibile."

#: musicbot/config.py:1165
#, python-format
msgid "Audio Cache will be stored in:  %s"
msgstr "La cache audio sarà memorizzata in:  %s"

#: musicbot/config.py:1176
#, python-format
msgid "Error while reading config options.\n\n"
"Problem:\n"
"  No bot Token was specified in the config options or environment.\n\n"
"Solution:\n"
"  Set the Token config option or set environment variable %(env_var)s with an App token."
msgstr "Errore durante la lettura delle opzioni di configurazione.\n\n"
"Problema:\n"
"  Nessun bot Token è stato specificato nelle opzioni di configurazione o ambiente.\n\n"
"Soluzione:\n"
"  Imposta l'opzione di configurazione Token o imposta la variabile di ambiente %(env_var)s con un token App."

#: musicbot/config.py:1197
msgid "StatusMessage config option is too long, it will be limited to 128 characters."
msgstr "L'opzione di configurazione statusMessage è troppo lunga, sarà limitata a 128 caratteri."

#: musicbot/config.py:1206
#, python-format
msgid "The default playback speed must be between 0.5 and 100.0. The option value of %.3f will be limited instead."
msgstr "La velocità di riproduzione predefinita deve essere compresa tra 0.5 e 100.0. Il valore di opzione di %.3f sarà limitato."

#: musicbot/config.py:1216
msgid "Cookies TXT file detected. MusicBot will pass them to yt-dlp.\n"
"Cookies are not recommended, may not be supported, and may totally break.\n"
"Copying cookies from your web-browser risks exposing personal data and \n"
"in the best case can result in your accounts being banned!\n\n"
"You have been warned!  Good Luck!  \\U0001F596\n"
msgstr "Rilevato file TXT cookie. MusicBot li passerà a yt-dlp.\n"
"I cookie non sono raccomandati, potrebbero non essere supportati e potrebbero rompersi completamente.\n"
"La copia dei cookie dal tuo browser web rischia di esporre i dati personali e \n"
"nel migliore dei casi può comportare il divieto dei tuoi account!\n\n"
"Sei stato avvisato! Buona Fortuna!  \\U0001F596\n"

#: musicbot/config.py:1234
msgid "Validating options with service data..."
msgstr "Validazione delle opzioni con i dati del servizio..."

#: musicbot/config.py:1240
msgid "Acquired owner ID via API"
msgstr "ID proprietario acquisito tramite API"

#: musicbot/config.py:1244
msgid "Error while fetching 'OwnerID' automatically.\n\n"
"Problem:\n"
"  Discord App info is not available.\n"
"  This could be a temporary API outage or a bug.\n\n"
"Solution:\n"
"  Manually set the 'OwnerID' config option or try again later."
msgstr "Errore durante il recupero automatico di 'OwnerID'.\n\n"
"Problema:\n"
"  Le informazioni Discord App non sono disponibili.\n"
"  Potrebbe trattarsi di un'interruzione temporanea delle API o di un bug. Soluzione\n\n"
":\n"
"  Imposta manualmente l'opzione di configurazione 'OwnerID' o riprova più tardi."

#: musicbot/config.py:1256
msgid "MusicBot does not have a user instance, cannot proceed."
msgstr "MusicBot non ha un'istanza utente, non può procedere."

#: musicbot/config.py:1262
msgid "Error validating config options.\n\n"
"Problem:\n"
"  The 'OwnerID' config is the same as your Bot / App ID.\n\n"
"Solution:\n"
"  Do not use the Bot or App ID in the 'OwnerID' field."
msgstr "Errore nella convalida delle opzioni di configurazione.\n\n"
"Problema:\n"
"  La configurazione 'OwnerID' è la stessa del tuo Bot / ID app.\n\n"
"Soluzione:\n"
"  Non utilizzare il Bot o l'App ID nel campo 'OwnerID'."

#: musicbot/config.py:1286
msgid "Config options file not found. Checking for alternatives..."
msgstr "File delle opzioni di configurazione non trovato. Verifica la presenza di alternative..."

#: musicbot/config.py:1298
#, python-format
msgid "Renaming %(ini_file)s to %(option_file)s, you should probably turn file extensions on."
msgstr "Rinomina %(ini_file)s in %(option_file)s, probabilmente dovresti attivare le estensioni dei file."

#: musicbot/config.py:1306
#, python-format
msgid "Copying existing example options file:  %(example_file)s"
msgstr "Copia del file delle opzioni di esempio esistenti:  %(example_file)s"

#: musicbot/config.py:1315
#, python-format
msgid "Generated a new %(example_file)s and copied it to %(option_file)s"
msgstr "Generato un nuovo %(example_file)s e copiato in %(option_file)s"

#: musicbot/config.py:1323
msgid "Something went wrong while trying to find a config option file."
msgstr "Qualcosa è andato storto durante il tentativo di trovare un file di opzione di configurazione."

#: musicbot/config.py:1327
#, python-format
msgid "Error locating config.\n\n"
"Problem:\n"
"  Could not find or create a config file due to an error:\n"
"  %(raw_error)s\n\n"
"Solution:\n"
"  Verify the config folder and files exist and can be read by MusicBot."
msgstr "Errore nella localizzazione della configurazione.\n\n"
"Problema:\n"
"  Impossibile trovare o creare un file di configurazione a causa di un errore:\n"
"  %(raw_error)s\n\n"
"Soluzione:\n"
"  Verifica che la cartella di configurazione e i file esistano e possono essere letti da MusicBot."

#: musicbot/config.py:1345
#, python-format
msgid "Error loading config.\n\n"
"Problem:\n"
"  MusicBot could not read config file due to an error:\n"
"  %(raw_error)s\n\n"
"Solution:\n"
"  Make sure the file is accessible and error free.\n"
"  Copy the example file from the repo if all else fails."
msgstr "Errore nel caricamento della configurazione.\n\n"
"Problema:\n"
"  MusicBot non ha potuto leggere il file di configurazione a causa di un errore:\n"
"  %(raw_error)s\n\n"
"Soluzione:\n"
"  Assicurarsi che il file sia accessibile e privo di errori.\n"
"  Copiare il file di esempio dal repo se tutto il resto fallisce."

#: musicbot/config.py:1369
msgid "Dev Bug! Config option has getter that is not available."
msgstr "Dev Bug! L'opzione di configurazione ha getter che non è disponibile."

#: musicbot/config.py:1374
msgid "Dev Bug! Config option has invalid type, getter and default must be the same type."
msgstr "Dev Bug! L'opzione di configurazione ha tipo non valido, getter e predefinito deve essere lo stesso tipo."

#: musicbot/config.py:1394
msgid "Option was missing previously."
msgstr "L'opzione mancava in precedenza."

#: musicbot/config.py:1409
#, python-format
msgid "Config section not in parsed config! Missing: %s"
msgstr "Sezione di configurazione non in configurazione analizzata! Mancante: %s"

#: musicbot/config.py:1414
#, python-format
msgid "Saved config option: %(config)s  =  %(value)s"
msgstr "Opzione di configurazione salvata: %(config)s  =  %(value)s"

#: musicbot/config.py:1427
#, python-format
msgid "Failed to save config:  %s"
msgstr "Salvataggio della configurazione non riuscito:  %s"

#: musicbot/config.py:1932
msgid "Option names are not unique between INI sections!  Resolver is disabled."
msgstr "I nomi delle opzioni non sono univoci tra le sezioni INI! Resolver è disabilitato."

#: musicbot/config.py:2094 musicbot/permissions.py:859
#, python-format
msgid "Failed to save default INI file at:  %s"
msgstr "Impossibile salvare il file INI predefinito in:  %s"

#: musicbot/config.py:2184
#, python-format
msgid "Error loading config value.\n\n"
"Problem:\n"
"  The owner ID in [%(section)s] > %(option)s is not valid.\n\n"
"Solution:\n"
"  Set %(option)s to a numerical ID or set it to `auto` or `0` for automatic owner binding."
msgstr "Errore nel caricamento del valore di configurazione.\n\n"
"Problema:\n"
"  L'ID proprietario in [%(section)s] > %(option)s non è valido.\n\n"
"Soluzione:\n"
"  Imposta %(option)s su un ID numerico o impostalo su `auto` o `0` per il binding automatico del proprietario."

#: musicbot/config.py:2220
#, python-format
msgid "Error loading config value.\n\n"
"Problem:\n"
"  The config option [%(section)s] > %(option)s is not a valid file location.\n\n"
"Solution:\n"
"  Check the path setting and make sure the file exists and is accessible to MusicBot."
msgstr "Errore nel caricamento del valore di configurazione.\n\n"
"Problema:\n"
"  L'opzione di configurazione [%(section)s] > %(option)s non è una posizione valida del file.\n\n"
"Soluzione:\n"
"  Controlla l'impostazione del percorso e assicurati che il file esista ed sia accessibile a MusicBot."

#: musicbot/config.py:2250
#, python-format
msgid "Error loading config value.\n\n"
"Problem:\n"
"  One of the IDs in option [%(section)s] > %(option)s is invalid.\n\n"
"Solution:\n"
"  Ensure all IDs are numerical, and separated only by spaces or commas."
msgstr "Errore nel caricamento del valore di configurazione.\n\n"
"Problema:\n"
"  Uno degli ID nell'opzione [%(section)s] > %(option)s non è valido.\n\n"
"Soluzione:\n"
"  Assicurati che tutti gli ID siano numerici e separati solo da spazi o virgole."

#: musicbot/config.py:2283
#, python-format
msgid "Invalid DebugLevel option \"%(value)s\" given, falling back to level: %(fallback)s"
msgstr "Opzione DebugLevel \"%(value)s\" non valida, tornando al livello: %(fallback)s"

#: musicbot/config.py:2304
#, python-format
msgid "Option [%(section)s] > %(option)s has invalid config value '%(value)s' using default instead."
msgstr "Opzione [%(section)s] > %(option)s ha valore di configurazione non valido '%(value)s' usando invece predefinito."

#: musicbot/config.py:2353
#, python-format
msgid "Option [%(section)s] > %(option)s has a value greater than 100 %% (%(value)s) and will be set to %(fallback)s instead."
msgstr "Opzione [%(section)s] > %(option)s ha un valore maggiore di 100 %% (%(value)s) e sarà invece impostato su %(fallback)s."

#: musicbot/config.py:2437
#, python-format
msgid "Renaming INI file entry [%(old_s)s] > %(old_o)s  to  [%(new_s)s] > %(new_o)s"
msgstr "Rinomina la voce del file INI [%(old_s)s] > %(old_o)s  in [%(new_s)s] > %(new_o)s"

#: musicbot/config.py:2493
msgid "Upgrading config file with renamed options..."
msgstr "Aggiornamento file di configurazione con opzioni rinominate..."

#: musicbot/config.py:2510
msgid "Failed to upgrade config.  You'll need to upgrade it manually."
msgstr "Impossibile aggiornare la configurazione. Dovrai aggiornarla manualmente."

#: musicbot/config.py:2551
#, python-format
msgid "Block list file not found:  %s"
msgstr "File elenco blocco non trovato:  %s"

#: musicbot/config.py:2572
#, python-format
msgid "Could not load block list from file:  %s"
msgstr "Impossibile caricare l'elenco dei blocchi dal file:  %s"

#: musicbot/config.py:2610 musicbot/config.py:2643
#, python-format
msgid "Could not update the block list file:  %s"
msgstr "Impossibile aggiornare il file elenco blocchi:  %s"

#: musicbot/config.py:2670
#, python-format
msgid "Loaded User Block list with %s entries."
msgstr "Lista Blocco utente caricata con voci %s."

#: musicbot/config.py:2681
#, python-format
msgid "We found a legacy blacklist file, it will be renamed to:  %s"
msgstr "Abbiamo trovato un file di blacklist legacy, che verrà rinominato in:  %s"

#: musicbot/config.py:2727
#, python-format
msgid "Loaded a Song Block list with %s entries."
msgstr "Caricato un elenco di blocchi brani con voci %s."

#: musicbot/constructs.py:221
msgid "Cannot load data for guild with ID 0. This is likely a bug in the code!"
msgstr "Impossibile caricare i dati per la gilda con ID 0. Probabilmente è un bug nel codice!"

#: musicbot/constructs.py:230
#, python-format
msgid "No file for guild %(id)s/%(name)s"
msgstr "Nessun file per la gilda %(id)s/%(name)s"

#: musicbot/constructs.py:239
#, python-format
msgid "Loading guild data for guild with ID:  %(id)s/%(name)s"
msgstr "Caricamento dei dati della gilda con ID:  %(id)s/%(name)s"

#: musicbot/constructs.py:246
#, python-format
msgid "An OS error prevented reading guild data file:  %s"
msgstr "Un errore OS ha impedito di leggere il file dati della gilda:  %s"

#: musicbot/constructs.py:255
#, python-format
msgid "Guild %(id)s/%(name)s has custom command prefix: %(prefix)s"
msgstr "La Gilda %(id)s/%(name)s ha il prefisso del comando personalizzato: %(prefix)s"

#: musicbot/constructs.py:275
msgid "Cannot save data for guild with ID 0. This is likely a bug in the code!"
msgstr "Impossibile salvare i dati per la gilda con ID 0. Probabilmente è un bug nel codice!"

#: musicbot/constructs.py:298
msgid "Could not save guild specific data due to OS Error."
msgstr "Impossibile salvare i dati specifici della gilda a causa di un errore del sistema."

#: musicbot/constructs.py:301
msgid "Failed to serialize guild specific data due to invalid data."
msgstr "Impossibile serializzare i dati specifici della gilda a causa di dati non validi."

#: musicbot/downloader.py:94
#, python-format
msgid "Forcing YTDLP to use User Agent:  %s"
msgstr "Forzare YTDLP per utilizzare l'agente utente:  %s"

#: musicbot/downloader.py:105
#, python-format
msgid "MusicBot will use cookies for yt-dlp from:  %s"
msgstr "MusicBot utilizzerà i cookie per yt-dlp da:  %s"

#: musicbot/downloader.py:111
msgid "Yt-dlp will use your configured proxy server."
msgstr "Yt-dlp userà il tuo server proxy configurato."

#: musicbot/downloader.py:134
msgid "Original OAuth2 plugin is installed and will be used instead.\n"
"This may cause MusicBot to not close completely, or hang pending authorization!\n"
"To close MusicBot, you must manually Kill the MusicBot process!\n"
"Yt-dlp is being set to show warnings and other log messages, to show the Auth code.\n"
"Uninstall the yt-dlp-youtube-oauth2 package to use integrated OAuth2 features instead."
msgstr "Il plugin originale OAuth2 è installato e verrà utilizzato al suo posto.\n"
"Questo potrebbe far sì che MusicBot non si chiuda completamente, o appenda l'autorizzazione in sospeso!\n"
"Per chiudere MusicBot, devi uccidere manualmente il processo MusicBot!\n"
"Yt-dlp è impostato per mostrare avvisi e altri messaggi di registro, per mostrare il codice Auth.\n"
"Disinstalla il pacchetto yt-dlp-youtube-oauth2 per utilizzare invece le funzionalità integrate di OAuth2."

#: musicbot/downloader.py:232
msgid "HEAD seems to have no headers..."
msgstr "HEAD sembra non avere intestazioni..."

#: musicbot/downloader.py:243
msgid "Checking media headers failed due to timeout."
msgstr "Controllo intestazioni multimediali non riuscito a causa del timeout."

#: musicbot/downloader.py:246
#, python-format
msgid "Failed HEAD request for:  %s"
msgstr "Richiesta HEAD non riuscita per:  %s"

#: musicbot/downloader.py:247
msgid "HEAD Request exception: "
msgstr "Richiesta di eccezione HEAD: "

#: musicbot/downloader.py:315
#, python-format
msgid "Sanitized YTDL Extraction Info (not JSON):\n"
"%s"
msgstr "Informazioni Di Estrazione Ytdl Sanitizzate (non Json):\n"
"%s"

#: musicbot/downloader.py:317
#, python-format
msgid "Sanitized YTDL Extraction Info (not JSON):  %s"
msgstr "Informazioni Di Estrazione Ytdl Sanitizzate (non Json):  %s"

#: musicbot/downloader.py:381
msgid "Song info extraction returned no data."
msgstr "L'estrazione di informazioni brano non ha restituito dati."

#: musicbot/downloader.py:437
#, python-format
msgid "Called extract_info with:  '%(subject)s', %(args)s, %(kws)s"
msgstr "Chiamato extract_info con: '%(subject)s', %(args)s, %(kws)s"

#: musicbot/downloader.py:445
msgid "Cannot run extraction, loop is closed. (This is normal on shutdowns.)"
msgstr "Impossibile eseguire l'estrazione, il ciclo è chiuso. (Questo è normale in caso di arresti)"

#: musicbot/downloader.py:447
msgid "Cannot continue extraction, event loop is closed."
msgstr "Impossibile continuare l'estrazione, il ciclo dell'evento è chiuso."

#: musicbot/downloader.py:456
msgid "Spotify URL is invalid or not supported."
msgstr "Spotify URL non è valido o non supportato."

#: musicbot/downloader.py:489 musicbot/downloader.py:510
#, python-format
msgid "Error in yt-dlp while downloading data: %(raw_error)s"
msgstr "Errore in yt-dlp durante il download dei dati: %(raw_error)s"

#: musicbot/downloader.py:493
msgid "Download Error with stream URL"
msgstr "Errore di download con URL dello stream"

#: musicbot/downloader.py:498
msgid "Assuming content is a direct stream"
msgstr "Supponendo che il contenuto sia uno stream diretto"

#: musicbot/downloader.py:506
msgid "Cannot stream an invalid URL."
msgstr "Impossibile trasmettere un URL non valido."

#: musicbot/downloader.py:517
msgid "Caught NoSupportingHandlers, trying again after replacing colon with space."
msgstr "Catturato NoSupportingHandlers, riprovando dopo aver sostituito colon con spazio."

#: musicbot/downloader.py:544
msgid "Extractor youtube:search returned single-entry result, replacing base info with entry info."
msgstr "Estrattore youtube:search ha restituito il risultato di un singolo ingresso, sostituendo le informazioni di base con le informazioni di entrata."

#: musicbot/downloader.py:560
#, python-format
msgid "Called safe_extract_info with:  %(args)s, %(kws)s"
msgstr "Chiamato safe_extract_info con:  %(args)s, %(kws)s"

#: musicbot/downloader.py:580
msgid "The local media file could not be found."
msgstr "Impossibile trovare il file multimediale locale."

#: musicbot/downloader.py:618
msgid "Missing __input_subject from YtdlpResponseDict"
msgstr "Manca __input_subject da YtdlpResponseDict"

#: musicbot/downloader.py:623
msgid "Entries is not a list in YtdlpResponseDict, set process=True to avoid this."
msgstr "Le voci non sono una lista in YtdlpResponseDict, set process=True per evitare questo."

#: musicbot/downloader.py:852
#, python-format
msgid "Warning, duration error for: %(url)s"
msgstr "Attenzione, errore di durata per: %(url)s"

#: musicbot/entry.py:45
msgid "module 'pymediainfo' not found, will fall back to ffprobe."
msgstr "modulo 'pymediainfo' non trovato, cadrà di nuovo a ffprobe."

#: musicbot/entry.py:135
#, python-format
msgid "Created future for %r"
msgstr "Creato il futuro per %r"

#: musicbot/entry.py:147
#, python-format
msgid "Completed futures for %(entry)r with %(callback)r"
msgstr "Futures completati per %(entry)r con %(callback)r"

#: musicbot/entry.py:157
msgid "Unhandled exception in _for_each_future callback."
msgstr "Eccezione non gestita in _for_each_future callback."

#: musicbot/entry.py:188
#, python-format
msgid "Starting asyncio subprocess (%(process)s) with command: %(run)s"
msgstr "Avviare asyncio sottoprocesso (%(process)s) con comando: %(run)s"

#: musicbot/entry.py:223
#, python-format
msgid "Extraction did not provide a duration for this entry.\n"
"MusicBot cannot estimate queue times until it is downloaded.\n"
"Entry name:  %s"
msgstr "L'estrazione non ha fornito una durata per questa voce.\n"
"MusicBot non può stimare i tempi di coda finché non viene scaricata.\n"
"Nome voce: %s"

#: musicbot/entry.py:339 musicbot/entry.py:850 musicbot/entry.py:1069
msgid "Entry data is missing version number, cannot deserialize."
msgstr "I dati dell'elemento sono mancanti numero di versione, impossibile deserializzare."

#: musicbot/entry.py:342 musicbot/entry.py:853 musicbot/entry.py:1072
msgid "Entry data has the wrong version number, cannot deserialize."
msgstr "I dati dell'elemento hanno il numero di versione sbagliato, non è possibile deserializzare."

#: musicbot/entry.py:358
#, python-format
msgid "Deserialized URLPlaylistEntry cannot find channel with ID:  %s"
msgstr "L'URLPlaylistEntry deserializzato non può trovare il canale con ID:  %s"

#: musicbot/entry.py:374
#, python-format
msgid "Deserialized URLPlaylistEntry has the wrong channel type:  %s"
msgstr "L'URLPlaylistEntry deserializzato ha il tipo di canale sbagliato:  %s"

#: musicbot/entry.py:396
#, python-format
msgid "Deserialized URLPlaylistEntry cannot find author with ID:  %s"
msgstr "L'URLPlaylistEntry deserializzato non può trovare l'autore con ID:  %s"

#: musicbot/entry.py:402
msgid "Deserialized URLPlaylistEntry has an author ID but no channel for lookup!"
msgstr "URLPlaylistEntry deserializzato ha un ID autore ma nessun canale per la ricerca!"

#: musicbot/entry.py:412 musicbot/entry.py:919 musicbot/entry.py:1142
#, python-format
msgid "Could not load %s"
msgstr "Impossibile caricare %s"

#: musicbot/entry.py:449
#, python-format
msgid "Cannot download Spotify links, processing error with type: %(type)s"
msgstr "Impossibile scaricare i collegamenti di Spotify, errore di elaborazione con il tipo: %(type)s"

#: musicbot/entry.py:461 musicbot/entry.py:924 musicbot/entry.py:1175
#, python-format
msgid "Getting ready for entry:  %r"
msgstr "Preparazione per l'ingresso:  %r"

#: musicbot/entry.py:478
msgid "Download cached with different extension..."
msgstr "Scarica la cache con diverse estensioni..."

#: musicbot/entry.py:487
msgid "Local size different from remote size. Re-downloading..."
msgstr "Dimensione locale diversa dalla dimensione remota. Ricaricamento..."

#: musicbot/entry.py:491
#, python-format
msgid "Download already cached at:  %s"
msgstr "Download già memorizzato in:  %s"

#: musicbot/entry.py:511 musicbot/entry.py:1191
#, python-format
msgid "MusicBot could not get duration data for this entry.\n"
"Queue time estimation may be unavailable until this track is cleared.\n"
"Entry file: %s"
msgstr "MusicBot non può ottenere dati di durata per questa voce. La stima del tempo in coda\n"
"potrebbe non essere disponibile fino a quando questa traccia non sarà stata cancellata.\n"
"File di ingresso: %s"

#: musicbot/entry.py:518
#, python-format
msgid "Got duration of %(time)s seconds for file:  %(file)s"
msgstr "Durata di %(time)s secondi per il file:  %(file)s"

#: musicbot/entry.py:529 musicbot/entry.py:1209
msgid "There as a problem with working out EQ, likely caused by a strange installation of FFmpeg. This has not impacted the ability for the bot to work, but will mean your tracks will not be equalized."
msgstr "C'è un problema con il lavoro EQ, probabilmente causato da una strana installazione di FFmpeg. Questo non ha influenzato la capacità del bot di funzionare, ma significa che le tue tracce non saranno equalizzate."

#: musicbot/entry.py:541 musicbot/entry.py:1222
msgid "Exception while checking entry data."
msgstr "Eccezione durante il controllo dei dati di inserimento."

#: musicbot/entry.py:552 musicbot/entry.py:1233
#, python-format
msgid "Trying to get duration via pymediainfo for:  %s"
msgstr "Tentativo di ottenere durata tramite pymediainfo per:  %s"

#: musicbot/entry.py:558 musicbot/entry.py:1239
msgid "Failed to get duration via pymediainfo."
msgstr "Impossibile ottenere la durata tramite pymediainfo."

#: musicbot/entry.py:565 musicbot/entry.py:1246
#, python-format
msgid "Trying to get duration via ffprobe for:  %s"
msgstr "Tentativo di ottenere durata tramite ffprobe per:  %s"

#: musicbot/entry.py:568 musicbot/entry.py:1249
msgid "Could not locate ffprobe in your path!"
msgstr "Impossibile individuare ffprobe nel tuo percorso!"

#: musicbot/entry.py:589 musicbot/entry.py:1270
msgid "ffprobe returned something that could not be used."
msgstr "ffprobe ha restituito qualcosa che non poteva essere utilizzato."

#: musicbot/entry.py:592 musicbot/entry.py:1273
msgid "ffprobe could not be executed for some reason."
msgstr "ffprobe non può essere eseguito per qualche ragione."

#: musicbot/entry.py:602 musicbot/entry.py:1283
#, python-format
msgid "Calculating mean volume of:  %s"
msgstr "Calcolo del volume medio di:  %s"

#: musicbot/entry.py:605 musicbot/entry.py:1286
msgid "Could not locate ffmpeg on your path!"
msgstr "Impossibile individuare ffmpeg sul tuo percorso!"

#: musicbot/entry.py:631 musicbot/entry.py:1312
msgid "Could not parse 'I' in normalize json."
msgstr "Impossibile analizzare 'I' nel normalizzare json."

#: musicbot/entry.py:639 musicbot/entry.py:1320
msgid "Could not parse 'LRA' in normalize json."
msgstr "Impossibile analizzare 'LRA' nella normalizzazione json."

#: musicbot/entry.py:647 musicbot/entry.py:1328
msgid "Could not parse 'TP' in normalize json."
msgstr "Impossibile analizzare 'TP' nel normalizzare json."

#: musicbot/entry.py:655 musicbot/entry.py:1336
msgid "Could not parse 'thresh' in normalize json."
msgstr "Impossibile analizzare 'thresh' nella normalizzazione json."

#: musicbot/entry.py:663 musicbot/entry.py:1344
msgid "Could not parse 'offset' in normalize json."
msgstr "Impossibile analizzare 'offset' nel normalizzare json."

#: musicbot/entry.py:680
#, python-format
msgid "Download started:  %r"
msgstr "Download avviato:  %r"

#: musicbot/entry.py:685
#, python-format
msgid "Download attempt %s of 3..."
msgstr "Scarica tentativo %s di 3..."

#: musicbot/entry.py:697
#, python-format
msgid "Download incomplete, retrying in %(time).1f seconds.  Reason: %(raw_error)s"
msgstr "Download incompleto, riprovando in %(time).1f secondi. Motivo: %(raw_error)s"

#: musicbot/entry.py:705
#, python-format
msgid "Download failed, not retrying! Reason:  %(raw_error)s"
msgstr "Download fallito, non riprovare! Motivo:  %(raw_error)s"

#: musicbot/entry.py:710
#, python-format
msgid "Download did not complete due to an error: %(raw_error)s"
msgstr "Il download non è stato completato a causa di un errore: %(raw_error)s"

#: musicbot/entry.py:716
#, python-format
msgid "Download failed due to a yt-dlp error: %(raw_error)s"
msgstr "Download fallito a causa di un errore yt-dlp: %(raw_error)s"

#: musicbot/entry.py:722
msgid "Extraction encountered an unhandled exception."
msgstr "L'estrazione ha incontrato un'eccezione non gestita."

#: musicbot/entry.py:726
#, python-format
msgid "Download failed due to an unhandled exception: %(raw_error)s"
msgstr "Download fallito a causa di un'eccezione non gestita: %(raw_error)s"

#: musicbot/entry.py:731
#, python-format
msgid "Download failed:  %r"
msgstr "Download non riuscito:  %r"

#: musicbot/entry.py:732
msgid "Failed to extract data for the requested media."
msgstr "Estrazione dei dati per il supporto richiesto non riuscita."

#: musicbot/entry.py:734
#, python-format
msgid "Download complete:  %r"
msgstr "Download completato:  %r"

#: musicbot/entry.py:866
#, python-format
msgid "Deserialized StreamPlaylistEntry cannot find channel with ID:  %s"
msgstr "StreamPlaylistEntry deserializzato non può trovare il canale con ID:  %s"

#: musicbot/entry.py:882
#, python-format
msgid "Deserialized StreamPlaylistEntry has the wrong channel type:  %s"
msgstr "StreamPlaylistEntry deserializzato ha il tipo di canale sbagliato:  %s"

#: musicbot/entry.py:904
#, python-format
msgid "Deserialized StreamPlaylistEntry cannot find author with ID:  %s"
msgstr "StreamPlaylistEntry deserializzato non può trovare l'autore con ID:  %s"

#: musicbot/entry.py:910
msgid "Deserialized StreamPlaylistEntry has an author ID but no channel for lookup!"
msgstr "Deserialized StreamPlaylistEntry ha un ID autore ma nessun canale per la ricerca!"

#: musicbot/entry.py:1088
#, python-format
msgid "Deserialized LocalFilePlaylistEntry cannot find channel with ID:  %s"
msgstr "LocalFilePlaylistEntry deserializzato non riesce a trovare il canale con ID:  %s"

#: musicbot/entry.py:1104
#, python-format
msgid "Deserialized LocalFilePlaylistEntry has the wrong channel type:  %s"
msgstr "Deserializzato LocalFilePlaylistEntry ha il tipo di canale sbagliato:  %s"

#: musicbot/entry.py:1126
#, python-format
msgid "Deserialized LocalFilePlaylistEntry cannot find author with ID:  %s"
msgstr "Deserialized LocalFilePlaylistEntry non può trovare l'autore con ID:  %s"

#: musicbot/entry.py:1132
msgid "Deserialized LocalFilePlaylistEntry has an author ID but no channel for lookup!"
msgstr "Deserialized LocalFilePlaylistEntry ha un ID autore ma nessun canale per la ricerca!"

#: musicbot/entry.py:1198
#, python-format
msgid "Got duration of %(seconds)s seconds for file:  %(file)s"
msgstr "Durata di %(seconds)s secondi per il file:  %(file)s"

#: musicbot/filecache.py:118
#, python-format
msgid "Failed to delete cache file:  %s"
msgstr "Eliminazione del file cache non riuscita:  %s"

#: musicbot/filecache.py:129
msgid "Audio cache directory has been removed."
msgstr "La directory della cache audio è stata rimossa."

#: musicbot/filecache.py:136
msgid "Audio cache directory could not be removed or renamed."
msgstr "La directory della cache audio non può essere rimossa o rinominata."

#: musicbot/filecache.py:143
msgid "Audio cache directory could not be removed."
msgstr "La directory della cache audio non può essere rimossa."

#: musicbot/filecache.py:152
msgid "Audio cache has no limits set, nothing to delete."
msgstr "La cache audio non ha limiti impostati, niente da eliminare."

#: musicbot/filecache.py:215
#, python-format
msgid "Audio cache deleted %(number)s file(s), total of %(size)s removed."
msgstr "Cache audio cancellato %(number)s file(s), totale %(size)s rimosso."

#: musicbot/filecache.py:223
#, python-format
msgid "Audio cached retained %(number)s file(s) from autoplaylist, total of %(size)s retained."
msgstr "File memorizzati nella cache %(number)s dalla playlist automatica, totale di %(size)s mantenuto."

#: musicbot/filecache.py:232
#, python-format
msgid "Audio cache is now %(size)s over %(number)s file(s)."
msgstr "La cache audio ora è %(size)s su %(number)s file."

#: musicbot/filecache.py:248
msgid "Audio cache directory is missing, nothing to delete."
msgstr "Manca la directory della cache audio, niente da eliminare."

#: musicbot/filecache.py:267
msgid "Audio cache file is from autoplaylist but marked as busted, ignoring it."
msgstr "Il file cache audio è dalla scaletta automatica, ma contrassegnato come busted, ignorandolo."

#: musicbot/filecache.py:278
#, python-format
msgid "Cache level requires cleanup. %s"
msgstr "Il livello della cache richiede la pulizia. %s"

#: musicbot/filecache.py:302
msgid "Auto playlist has no cache map, moving on."
msgstr "La playlist automatica non ha mappa cache, in movimento."

#: musicbot/filecache.py:310
#, python-format
msgid "Loaded auto playlist cache map with %s entries."
msgstr "Mappa cache playlist automatica caricata con voci %s."

#: musicbot/filecache.py:314
msgid "Failed to load auto playlist cache map."
msgstr "Impossibile caricare la mappa della cache della playlist automatica."

#: musicbot/filecache.py:333
#, python-format
msgid "Saved auto playlist cache map with %s entries."
msgstr "Mappa della cache della scaletta automatica salvata con voci %s."

#: musicbot/filecache.py:337
msgid "Failed to save auto playlist cache map."
msgstr "Impossibile salvare la mappa della cache della playlist automatica."

#: musicbot/filecache.py:355
#, python-format
msgid "Auto playlist cache map conflict on Key: %(file)s  Old: %(old)s  New: %(new)s"
msgstr "Conflitto della mappa della cache della scaletta automatica sulla chiave: %(file)s  Vecchio: %(old)s  Nuovo: %(new)s"

#: musicbot/i18n.py:229
#, python-format
msgid "Lang Argument Error:  %s"
msgstr "Errore Argomento Lang:  %s"

#: musicbot/i18n.py:290
#, python-format
msgid "Failed to load log translations for any of:  [%s]  in:  %s"
msgstr "Impossibile caricare le traduzioni del log per uno qualsiasi di:  [%s]  in:  %s"

#: musicbot/i18n.py:333
#, python-format
msgid "Failed to load discord translations for any of:  [%s]  in:  %s"
msgstr "Caricamento delle traduzioni discord non riuscito per uno qualsiasi di:  [%s]  in:  %s"

#: musicbot/json.py:14
#, python-format
msgid "Loading JSON file: %s"
msgstr "Caricamento file JSON: %s"

#: musicbot/json.py:27
#, python-format
msgid "Error parsing %s as JSON"
msgstr "Errore nell'analizzare %s come JSON"

#: musicbot/json.py:36
#, python-format
msgid "Could not grab data from JSON key: %s"
msgstr "Impossibile catturare i dati dalla chiave JSON: %s"

#: musicbot/logs.py:124
msgid "Skipping logger setup, already set up"
msgstr "Impossibile impostare il logger, già impostato"

#: musicbot/logs.py:276
#, python-format
msgid "Log level was previously set via override to: %s"
msgstr "Il livello di log è stato precedentemente impostato tramite override a: %s"

#: musicbot/logs.py:285
#, python-format
msgid "Changing log level to:  %s"
msgstr "Cambiare il livello del log in:  %s"

#: musicbot/logs.py:314
msgid "MusicBot loggers have been called to shut down."
msgstr "I logger di MusicBot sono stati chiamati a chiudere."

#: musicbot/permissions.py:118
#, python-format
msgid "Permissions file not found, copying from:  %s"
msgstr "File di permessi non trovato, copia da:  %s"

#: musicbot/permissions.py:128
#, python-format
msgid "Error copying example permissions file:  %s"
msgstr "Errore nel copiare il file dei permessi di esempio:  %s"

#: musicbot/permissions.py:165
msgid "Config 'OwnerID' is set auto, will set correctly later."
msgstr "La configurazione 'OwnerID' è impostata automaticamente, verrà impostata correttamente in seguito."

#: musicbot/permissions.py:182
msgid "Validating permissions..."
msgstr "Convalida dei permessi..."

#: musicbot/permissions.py:184
msgid "Setting auto 'OwnerID' for owner permissions group."
msgstr "Impostazione automatica 'OwnerID' per il gruppo di permessi del proprietario."

#: musicbot/permissions.py:238
#, python-format
msgid "Updating group in permissions file:  %s"
msgstr "Aggiornamento del gruppo nel file autorizzazioni:  %s"

#: musicbot/permissions.py:244
#, python-format
msgid "Deleting group from permissions file:  %s"
msgstr "Eliminazione gruppo dal file permessi:  %s"

#: musicbot/permissions.py:249
#, python-format
msgid "Adding new group to permissions file:  %s"
msgstr "Aggiunta di un nuovo gruppo al file autorizzazioni:  %s"

#: musicbot/permissions.py:264
msgid "Saving permissions file now."
msgstr "Salvataggio file permessi ora."

#: musicbot/permissions.py:270
msgid "ConfigUpdater could not parse the permissions file!"
msgstr "ConfigUpdater non può analizzare il file dei permessi!"

#: musicbot/permissions.py:272
msgid "You have a duplicate section, fix your Permissions file!"
msgstr "Hai una sezione duplicata, correggi il tuo file Permessi!"

#: musicbot/permissions.py:274
#, python-format
msgid "Failed to save permissions group:  %s"
msgstr "Salvataggio del gruppo di permessi non riuscito:  %s"

#: musicbot/permissions.py:289
msgid "Dev Bug! Permission has getter that is not available."
msgstr "Dev Bug! Il permesso ha getter che non è disponibile."

#: musicbot/permissions.py:294
msgid "Dev Bug! Permission has invalid type, getter and default must be the same type."
msgstr "Dev Bug! Il permesso ha un tipo, getter e predefinito non validi devono essere lo stesso tipo."

#: musicbot/permissions.py:535
msgid "Max search items can't be larger than 100. Setting to 100."
msgstr "Gli elementi di ricerca massimi non possono essere più grandi di 100. Impostazione a 100."

#: musicbot/permissions.py:613
#, python-format
msgid "You do not have permission to play the requested media.\n"
"The yt-dlp extractor `%(extractor)s` is not permitted in your group."
msgstr "Non hai il permesso di riprodurre il supporto richiesto.\n"
"L'estrattore yt-dlp `%(extractor)s` non è permesso nel tuo gruppo."

#: musicbot/player.py:85
#, python-format
msgid "Cleanup got called on the audio source:  %r"
msgstr "Cleanup è stato chiamato sulla sorgente audio:  %r"

#: musicbot/player.py:190
#, python-format
msgid "MusicPlayer.skip() is called:  %s"
msgstr "MusicPlayer.skip() è chiamato:  %s"

#: musicbot/player.py:200
#, python-format
msgid "MusicPlayer.stop() is called:  %s"
msgstr "MusicPlayer.stop() è chiamato:  %s"

#: musicbot/player.py:215
msgid "Guild or network unavailable, cannot resume playback."
msgstr "Gilda o rete non disponibile, impossibile riprendere la riproduzione."

#: musicbot/player.py:219
#, python-format
msgid "MusicPlayer.resume() is called:  %s"
msgstr "MusicPlayer.resume() è chiamato:  %s"

#: musicbot/player.py:239
#, python-format
msgid "MusicPlayer.pause() is called:  %s"
msgstr "MusicPlayer.pause() è chiamato:  %s"

#: musicbot/player.py:261
#, python-format
msgid "MusicPlayer.kill() is called:  %s"
msgstr "MusicPlayer.kill() è chiamato:  %s"

#: musicbot/player.py:287
msgid "Playback finished, but _current_entry is None."
msgstr "Riproduzione terminata, ma _current_entry non è nessuno."

#: musicbot/player.py:342
msgid "Possible Warning from kill_current_player()"
msgstr "Possibile avviso da kill_current_player()"

#: musicbot/player.py:361
#, python-format
msgid "MusicPlayer.play() is called:  %s"
msgstr "MusicPlayer.play() è chiamato:  %s"

#: musicbot/player.py:371
msgid "MusicPlayer is dead, cannot play."
msgstr "MusicPlayer è morto, non può giocare."

#: musicbot/player.py:376
msgid "Guild or network unavailable, cannot start playback."
msgstr "Gilda o rete non disponibile, impossibile avviare la riproduzione."

#: musicbot/player.py:381
msgid "MusicPlayer was previously paused, resuming current player."
msgstr "MusicPlayer è stato precedentemente in pausa, riprendendo il giocatore corrente."

#: musicbot/player.py:387
msgid "MusicPlayer already locked for playback, this call is ignored."
msgstr "MusicPlayer già bloccato per la riproduzione, questa chiamata è ignorata."

#: musicbot/player.py:398
msgid "Failed to get next entry."
msgstr "Impossibile ottenere la prossima voce."

#: musicbot/player.py:402
msgid "Failed to process entry for playback."
msgstr "Impossibile elaborare la voce per la riproduzione."

#: musicbot/player.py:453
#, python-format
msgid "Creating player with options: ffmpeg %(before)s -i %(input)s %(after)s"
msgstr "Creazione giocatore con opzioni: ffmpeg %(before)s -i %(input)s %(after)s"

#: musicbot/player.py:457
#, python-format
msgid "Playing %(source)r using %(client)r"
msgstr "Riproduzione di %(source)r usando %(client)r"

#: musicbot/player.py:488
#, python-format
msgid "Skipping deletion of '%s', found song in queue"
msgstr "Eliminazione saltata di '%s', canzone trovata in coda"

#: musicbot/player.py:492
#, python-format
msgid "Deleting file:  %s"
msgstr "Eliminazione del file:  %s"

#: musicbot/player.py:497
#, python-format
msgid "File deleted:  %s"
msgstr "File eliminato:  %s"

#: musicbot/player.py:501
msgid "Cannot delete file, it is currently in use."
msgstr "Impossibile eliminare il file, è attualmente in uso."

#: musicbot/player.py:504
msgid "Cannot delete file due to a permission error."
msgstr "Impossibile eliminare il file a causa di un errore di autorizzazione."

#: musicbot/player.py:509
msgid "Cannot delete file, it was not found."
msgstr "Impossibile eliminare il file, non è stato trovato."

#: musicbot/player.py:515
msgid "Error while trying to delete file."
msgstr "Errore durante il tentativo di eliminare il file."

#: musicbot/player.py:520
msgid "Could not delete file, giving up and moving on"
msgstr "Impossibile eliminare il file, arrendersi e spostarsi"

#: musicbot/player.py:581
#, python-format
msgid "Deserialize returned an object that is not a MusicPlayer:  %s"
msgstr "Deserialize ha restituito un oggetto che non è un MusicPlayer:  %s"

#: musicbot/player.py:586
msgid "Failed to deserialize player"
msgstr "Impossibile deserializzare il giocatore"

#: musicbot/player.py:648
#, python-format
msgid "Data from ffmpeg: %s"
msgstr "Dati da ffmpeg: %s"

#: musicbot/player.py:658
#, python-format
msgid "Error from ffmpeg: %s"
msgstr "Errore da ffmpeg: %s"

#: musicbot/player.py:666
#, python-format
msgid "Warning from ffmpeg:  %s"
msgstr "Attenzione da ffmpeg:  %s"

#: musicbot/player.py:695
msgid "Unknown error decoding message from ffmpeg"
msgstr "Messaggio di decodifica di errore sconosciuto da ffmpeg"

#: musicbot/player.py:699
#, python-format
msgid "Decoded data from ffmpeg: %s"
msgstr "Dati decodificati da ffmpeg: %s"

#: musicbot/playlist.py:121
#, python-format
msgid "Adding stream entry for URL:  %(url)s"
msgstr "Aggiunta voce stream per URL:  %(url)s"

#: musicbot/playlist.py:159
msgid "Could not extract information"
msgstr "Impossibile estrarre le informazioni"

#: musicbot/playlist.py:163
msgid "This is a playlist."
msgstr "Questa è una playlist."

#: musicbot/playlist.py:177
msgid "Entry info appears to be a stream, adding stream entry..."
msgstr "Le informazioni di immissione sembrano essere un flusso, aggiungendo voce di flusso..."

#: musicbot/playlist.py:195
#, python-format
msgid "Invalid content type `%(type)s` for URL: %(url)s"
msgstr "Tipo di contenuto non valido `%(type)s` per URL: %(url)s"

#: musicbot/playlist.py:203
msgid "Got text/html for content-type, this might be a stream."
msgstr "Ottenuto testo/html per il tipo di contenuto, questo potrebbe essere un flusso."

#: musicbot/playlist.py:210
#, python-format
msgid "Questionable content-type \"%(type)s\" for url:  %(url)s"
msgstr "Tipo di contenuto discutibile \"%(type)s\" per url:  %(url)s"

#: musicbot/playlist.py:215
#, python-format
msgid "Adding URLPlaylistEntry for: %(subject)s"
msgstr "Aggiunta di URLPlaylistEntry per: %(subject)s"

#: musicbot/playlist.py:235
#, python-format
msgid "Adding LocalFilePlaylistEntry for: %(subject)s"
msgstr "Aggiunta di LocalFilePlaylistEntry per: %(subject)s"

#: musicbot/playlist.py:280
#, python-format
msgid "Ignored video from compound playlist link with ID:  %s"
msgstr "Video ignorato dal collegamento composto alla scaletta con ID:  %s"

#: musicbot/playlist.py:292
#, python-format
msgid "Not allowing entry that is in song block list:  %(title)s  URL: %(url)s"
msgstr "Non consentire la voce che è nella lista dei brani bloccati:  %(title)s  URL: %(url)s"

#: musicbot/playlist.py:305
#, python-format
msgid "Ignoring song in entries by '%s', duration longer than permitted maximum."
msgstr "Ignorare il brano nelle voci di '%s', durata più lunga del massimo consentito."

#: musicbot/playlist.py:317
#, python-format
msgid "Not adding YouTube video because it is marked private or deleted:  %s"
msgstr "Non aggiungere video YouTube perché è contrassegnato privato o cancellato:  %s"

#: musicbot/playlist.py:342
msgid "Could not add item"
msgstr "Impossibile aggiungere l'elemento"

#: musicbot/playlist.py:343
#, python-format
msgid "Item: %s"
msgstr "Articolo: %s"

#: musicbot/playlist.py:346
#, python-format
msgid "Skipped %s bad entries"
msgstr "Voci saltate %s errate"

#: musicbot/playlist.py:387
msgid "Reorder looping over entries."
msgstr "Riordina il loop sopra le voci."

#: musicbot/playlist.py:469
#, python-format
msgid "Pre-downloading next track:  %r"
msgstr "Pre-scaricamento traccia successiva:  %r"

#: musicbot/playlist.py:495
msgid "no duration data"
msgstr "nessun dato di durata"

#: musicbot/playlist.py:504
msgid "no duration data in current entry"
msgstr "nessun dato di durata nella voce corrente"

#: musicbot/spotify.py:119
#, python-format
msgid "Invalid track_data, must be of type `track` got `%(type)s`"
msgstr "Track_data non valido, deve essere di tipo `track` ottenuto `%(type)s`"

#: musicbot/spotify.py:301
#, python-format
msgid "Ignored non-track entry in playlist with type:  %s"
msgstr "Elemento non tracciato ignorato nella scaletta con tipo:  %s"

#: musicbot/spotify.py:487
#, python-format
msgid "Spotify Album total tacks:  %(total)s  Next URL: %(url)s"
msgstr "Puntate totali dell'album Spotify:  %(total)s  URL successivo: %(url)s"

#: musicbot/spotify.py:492
#, python-format
msgid "Getting Spotify Album Next URL:  %s"
msgstr "Ottenere Spotify Album Prossimo URL:  %s"

#: musicbot/spotify.py:503
#, python-format
msgid "Spotify Album Object may not be complete, expected %(total)s tracks but got %(number)s"
msgstr "Spotify Album Object potrebbe non essere completo, atteso %(total)s brani ma ottenuto %(number)s"

#: musicbot/spotify.py:507
msgid "Spotify Album has more tracks than initial total."
msgstr "Spotify Album ha più tracce del totale iniziale."

#: musicbot/spotify.py:530
#, python-format
msgid "Spotify Playlist total tacks: %(total)s  Next URL: %(url)s"
msgstr "Punteggi totali della scaletta Spotify: %(total)s  URL successivo: %(url)s"

#: musicbot/spotify.py:535
#, python-format
msgid "Getting Spotify Playlist Next URL:  %s"
msgstr "Ottenere Playlist Spotify Prossimo URL:  %s"

#: musicbot/spotify.py:546
#, python-format
msgid "Spotify Playlist Object may not be complete, expected %(total)s tracks but got %(number)s"
msgstr "Spotify Playlist Object potrebbe non essere completo, atteso %(total)s brani ma ottenuto %(number)s"

#: musicbot/spotify.py:550
msgid "Spotify Playlist has more tracks than initial total."
msgstr "Spotify Playlist ha più tracce del totale iniziale."

#: musicbot/spotify.py:555
#, python-format
msgid "Spotify Playlist contained %s usable tracks."
msgstr "Spotify Playlist conteneva tracce utilizzabili %s."

#: musicbot/spotify.py:581 musicbot/spotify.py:617
#, python-format
msgid "Response status is not OK: [%(status)s] %(reason)s"
msgstr "Lo stato della risposta non va bene: [%(status)s] %(reason)s"

#: musicbot/spotify.py:587 musicbot/spotify.py:623 musicbot/spotify.py:733
msgid "Response JSON did not decode to a dict!"
msgstr "La risposta JSON non ha decodificato un detto!"

#: musicbot/spotify.py:596
#, python-format
msgid "Failed making GET request to url:  %s"
msgstr "Creazione della richiesta GET all'url non riuscita:  %s"

#: musicbot/spotify.py:602
#, python-format
msgid "Could not make GET to URL:  %(url)s  Reason:  %(raw_error)s"
msgstr "Impossibile rendere GET all'URL:  %(url)s  Motivo:  %(raw_error)s"

#: musicbot/spotify.py:632
#, python-format
msgid "Failed making POST request to url:  %s"
msgstr "Creazione della richiesta POST all'url non riuscita:  %s"

#: musicbot/spotify.py:638
#, python-format
msgid "Could not make POST to URL:  %(url)s  Reason:  %(raw_error)s"
msgstr "Impossibile rendere POST all'URL:  %(url)s  Motivo:  %(raw_error)s"

#: musicbot/spotify.py:672
msgid "Failed to get a guest token from Spotify, please try specifying client ID and client secret"
msgstr "Impossibile ottenere un token ospite da Spotify, si prega di provare a specificare l'ID client e il segreto client"

#: musicbot/spotify.py:679
msgid "Created a new Guest Mode access token."
msgstr "Creato un nuovo token di accesso in modalità ospite."

#: musicbot/spotify.py:683
#, python-format
msgid "API response did not contain the expected data. Missing key: %(raw_error)s"
msgstr "La risposta API non contiene i dati attesi. Chiave mancante: %(raw_error)s"

#: musicbot/spotify.py:689
#, python-format
msgid "API response contained unexpected data.\n"
"%(raw_error)s"
msgstr "La risposta API conteneva dati inattesi.\n"
"%(raw_error)s"

#: musicbot/spotify.py:696
msgid "Requested a token from Spotify, did not end up getting one"
msgstr "Richiesto un token da Spotify, non è finito per ottenere uno"

#: musicbot/spotify.py:700
msgid "Created a new Client Mode access token."
msgstr "Creato un nuovo token di accesso alla modalità client."

#: musicbot/spotify.py:727
#, python-format
msgid "API response status is not OK: [%(status)s]  %(reason)s"
msgstr "Lo stato della risposta API non va bene: [%(status)s]  %(reason)s"

#: musicbot/spotify.py:743
msgid "Failed to get Spotify Guest Token."
msgstr "Impossibile ottenere il Token di Spotify Ospite."

#: musicbot/spotify.py:750
#, python-format
msgid "Failed to get Guest Token due to: %(raw_error)s"
msgstr "Impossibile ottenere il Token ospite a causa di: %(raw_error)s"

#: musicbot/utils.py:174
msgid "Only the owner can use this command."
msgstr "Solo il proprietario può usare questo comando."

#: musicbot/utils.py:194
msgid "Only dev users can use this command."
msgstr "Solo gli utenti dev possono usare questo comando."

#: musicbot/utils.py:280 musicbot/utils.py:322
msgid "Cannot count members when voice_channel is None."
msgstr "Impossibile contare i membri quando voice_channel è Nessuno."

#: musicbot/ytdlp_oauth2_plugin.py:84
#, python-format
msgid "Failed to save ytdlp oauth2 token data due to:  %s"
msgstr "Impossibile salvare i dati del token ytdlp oauth2 a causa di:  %s"

#: musicbot/ytdlp_oauth2_plugin.py:91
msgid "Loading YouTube TV OAuth2 token data."
msgstr "Caricamento dei dati token OAuth2 di YouTube TV."

#: musicbot/ytdlp_oauth2_plugin.py:101
#, python-format
msgid "Failed to load ytdlp oauth2 token data due to:  %s"
msgstr "Caricamento dei dati del token ytdlp oauth2 non riuscito a causa di:  %s"

#: musicbot/ytdlp_oauth2_plugin.py:109
msgid "Storing YouTube TV OAuth2 token data"
msgstr "Memorizzazione dati token OAuth2 di YouTube TV"

#: musicbot/ytdlp_oauth2_plugin.py:139
msgid "Invalid cached OAuth2 token data."
msgstr "Dati token OAuth2 nella cache non validi."

#: musicbot/ytdlp_oauth2_plugin.py:154
msgid "Access token expired, refreshing"
msgstr "Token di accesso scaduto, aggiornamento"

#: musicbot/ytdlp_oauth2_plugin.py:175
msgid "YouTube cookies have been provided, but OAuth2 is being used. If you encounter problems, stop providing YouTube cookies to yt-dlp."
msgstr "I cookie di YouTube sono stati forniti, ma viene utilizzato OAuth2. In caso di problemi, smetta di fornire i cookie di YouTube a yt-dlp."

#: musicbot/ytdlp_oauth2_plugin.py:193
msgid "Refreshing YouTube TV oauth2 token..."
msgstr "Aggiornamento token di YouTube TV oauth2..."

#: musicbot/ytdlp_oauth2_plugin.py:211
#, python-format
msgid "Failed to refresh OAuth2 access token due to:  %s\n"
"Restarting authorization flow..."
msgstr "Impossibile aggiornare il token di accesso OAuth2 a causa di:  %s\n"
"Riavvio del flusso di autorizzazione..."

#: musicbot/ytdlp_oauth2_plugin.py:229
msgid "Starting oauth2 flow..."
msgstr "Avvio flusso oauth2..."

#: musicbot/ytdlp_oauth2_plugin.py:248
#, python-format
msgid "\n"
"NOTICE:\n"
"To give yt-dlp access to your account, visit:\n"
"  %s\n"
"Then enter this authorization code:  %s\n"
"You have %s seconds to complete authorization.\n"
msgstr "\n"
"AVVISO:\n"
"Per dare accesso yt-dlp al tuo account, visita:\n"
"  %s\n"
"Quindi inserisci questo codice di autorizzazione:  %s\n"
"Hai %s secondi per completare l'autorizzazione.\n"

#: musicbot/ytdlp_oauth2_plugin.py:257
msgid "The application may hang until authorization time out if closed at this point. This is normal."
msgstr "L'applicazione può essere sospesa fino al tempo di autorizzazione se chiusa a questo punto. Ciò è normale."

#: musicbot/ytdlp_oauth2_plugin.py:263
msgid "Timed out while waiting for OAuth2 token."
msgstr "Tempo scaduto durante l'attesa del token OAuth2."

#: musicbot/ytdlp_oauth2_plugin.py:291
msgid "The device code has expired, restarting authorization flow for yt-dlp."
msgstr "Il codice del dispositivo è scaduto, riavviare il flusso di autorizzazione per yt-dlp."

#: musicbot/ytdlp_oauth2_plugin.py:297
msgid "Yt-dlp OAuth2 authorization successful."
msgstr "Autorizzazione Yt-dlp OAuth2 riuscita."

#: musicbot/ytdlp_oauth2_plugin.py:327
#, python-format
msgid "Adding OAuth2 Plugin to Yt-dlp IE:  %s"
msgstr "Aggiunta del plugin OAuth2 a Yt-dlp IE:  %s"

#: musicbot/ytdlp_oauth2_plugin.py:349
#, python-format
msgid "Default Yt-dlp Clients:  %s"
msgstr "Client Yt-dlp Predefiniti:  %s"

#: run.py:99
msgid "Could not find git executable."
msgstr "Impossibile trovare git eseguibile."

#: run.py:148
msgid "Attempting to upgrade with `git pull` on current path."
msgstr "Tentativo di aggiornare con `git pull` sul percorso corrente."

#: run.py:155
#, python-format
msgid "Result of git pull:  %s"
msgstr "Risultato di git pull:  %s"

#: run.py:163
msgid "Upgrade failed, you need to run `git pull` manually."
msgstr "Aggiornamento non riuscito, è necessario eseguire `git pull` manualmente."

#: run.py:173
msgid "Cannot execute pip."
msgstr "Impossibile eseguire pip."

#: run.py:180
msgid "Error using -m method"
msgstr "Errore nell'utilizzo del metodo -m"

#: run.py:226
msgid "PIP failed while calling sub-process."
msgstr "PIP non è riuscito durante la chiamata del sottoprocesso."

#: run.py:229
msgid "PIP failed due to Permission Error."
msgstr "PIP non riuscito a causa di errore di autorizzazione."

#: run.py:233
#, python-format
msgid "PIP failed due to missing Python executable?  (%s)"
msgstr "PIP non riuscito a causa di Python eseguibile? (%s)"

#: run.py:238
msgid "PIP failed due to OSError."
msgstr "PIP non è riuscito a causa di OSError."

#: run.py:259
msgid "Could not decode pip update report JSON."
msgstr "Impossibile decodificare il rapporto di aggiornamento pip JSON."

#: run.py:281
msgid "Cannot locate or execute python -m pip"
msgstr "Impossibile individuare o eseguire python -m pip"

#: run.py:284
#, python-format
msgid "Attempting to upgrade with `%s` on current path..."
msgstr "Tentativo di aggiornare con `%s` sul percorso corrente..."

#: run.py:302
#, python-format
msgid "Result of pip upgrade:\n"
"%s"
msgstr "Risultato dell'aggiornamento dei pip:\n"
"%s"

#: run.py:307
#, python-format
msgid "Result exit code from pip upgrade: %s"
msgstr "Risultato codice di uscita dal pip upgrade: %s"

#: run.py:322
msgid "Upgrade failed to execute or we could not understand the output"
msgstr "Aggiornamento non riuscito a eseguire o non siamo riusciti a capire l'output"

#: run.py:325
#, python-format
msgid "You may need to run `%s` manually."
msgstr "Potrebbe essere necessario eseguire `%s` manualmente."

#: run.py:334
msgid "Press enter to continue . . ."
msgstr "Premere Invio per continuare . . ."

#: run.py:347
msgid "Starting sanity checks"
msgstr "Iniziare controlli di sanità"

#: run.py:361
msgid "Required checks passed."
msgstr "Controlli richiesti superati."

#: run.py:374
msgid "Skipped checking for updates."
msgstr "Controllo degli aggiornamenti saltato."

#: run.py:376
msgid "Optional checks passed."
msgstr "Controlli facoltativi superati."

#: run.py:384
msgid "Checking for Python 3.8+"
msgstr "Controllo per Python 3.8+"

#: run.py:388
#, python-format
msgid "Python 3.8+ is required. This version is %s"
msgstr "Python 3.8+ è obbligatorio. Questa versione è %s"

#: run.py:390
msgid "Attempting to locate Python 3.8..."
msgstr "Tentativo di localizzare Python 3.8..."

#: run.py:398
msgid "Could not locate py.exe"
msgstr "Impossibile individuare py.exe"

#: run.py:409
msgid "Could not execute `py.exe -3.8` "
msgstr "Impossibile eseguire `py.exe -3.8` "

#: run.py:413
msgid "Python 3 found.  Launching bot..."
msgstr "Python 3 trovato. Avvio bot..."

#: run.py:418
msgid "Trying \"python3.8\""
msgstr "Provare \"piton3.8\""

#: run.py:421
msgid "Could not locate python3.8 on path."
msgstr "Impossibile individuare python3.8 sul percorso."

#: run.py:435
#, python-format
msgid "\n"
"Python 3.8 found.  Re-launching bot using: %s run.py\n"
msgstr "\n"
"Python 3.8 trovato. Riavvio del bot utilizzando: %s run.py\n"

#: run.py:440
msgid "Could not find Python 3.8 or higher.  Please run the bot using Python 3.8"
msgstr "Impossibile trovare Python 3.8 o superiore. Esegui il bot usando Python 3.8"

#: run.py:473
msgid "Ensuring we're in the right environment"
msgstr "Garantire che ci troviamo nell'ambiente giusto"

#: run.py:498
#, python-format
msgid "Failed environment check, %s"
msgstr "Controllo ambiente non riuscito, %s"

#: run.py:509
msgid "Current working directory does not seem to be writable"
msgstr "La directory di lavoro corrente non sembra essere scrivibile"

#: run.py:510
msgid "Please move the bot to a folder that is writable"
msgstr "Spostare il bot in una cartella scrivibile"

#: run.py:519
#, python-format
msgid "Detected FFmpeg is installed at:  %s"
msgstr "FFmpeg rilevato è installato a:  %s"

#: run.py:521
msgid "Adding local bins/ folder environment PATH for bundled ffmpeg..."
msgstr "Aggiunta ambiente bins/ cartella locale PATH per ffmpeg..."

#: run.py:530
msgid "MusicBot could not locate FFmpeg binary in your environment.\n"
"Please install FFmpeg so it is available in your environment PATH variable."
msgstr "MusicBot non ha potuto individuare il binario FFmpeg nel tuo ambiente.\n"
"Si prega di installare FFmpeg in modo che sia disponibile nella variabile PATH del tuo ambiente."

#: run.py:535
msgid "On Windows, you can add a pre-compiled EXE to the MusicBot `bin` folder,\n"
"or you can install FFmpeg system-wide using WinGet or by running the install.bat file."
msgstr "Su Windows, puoi aggiungere una EXE pre-compilata alla cartella `bin` di MusicBot,\n"
"o è possibile installare FFmpeg a livello di sistema utilizzando WinGet o eseguendo l'installazione. al file."

#: run.py:540
msgid "On MacOS, you may be able to install FFmpeg via homebrew.\n"
"Otherwise, check the official FFmpeg site for build or install steps."
msgstr "Su MacOS, potresti essere in grado di installare FFmpeg tramite homebrew.\n"
"Altrimenti, controlla il sito ufficiale FFmpeg per creare o installare i passaggi."

#: run.py:545
msgid "On Linux, many distros make FFmpeg available via system package managers.\n"
"Check for ffmpeg with your system package manager or build from sources."
msgstr "Su Linux, molte distribuzioni rendono FFmpeg disponibile tramite i gestori dei pacchetti di sistema.\n"
"Verifica la presenza di ffmpeg con il gestore dei pacchetti di sistema o compila da sorgenti."

#: run.py:558
#, python-format
msgid "Less than %sMB of free space remains on this device"
msgstr "Rimangono meno di %sMB di spazio libero su questo dispositivo"

#: run.py:567
msgid "\n"
"Checking for updates to MusicBot or dependencies..."
msgstr "\n"
"Verifica aggiornamenti a MusicBot o dipendenze..."

#: run.py:584
msgid "No MusicBot updates available via `git` command."
msgstr "Nessun aggiornamento MusicBot disponibile tramite il comando `git`."

#: run.py:587
msgid "Could not check for updates using `git` commands.  You should check manually."
msgstr "Impossibile controllare gli aggiornamenti usando i comandi `git`. Dovresti controllare manualmente."

#: run.py:594
msgid "The following packages can be updated:\n"
msgstr "I seguenti pacchetti possono essere aggiornati:\n"

#: run.py:600
#, python-format
msgid "  %s  to version:  %s\n"
msgstr "  %s  alla versione:  %s\n"

#: run.py:611
msgid "No dependency updates available via `pip` command."
msgstr "Nessun aggiornamento delle dipendenze disponibile tramite il comando `pip`."

#: run.py:614
msgid "Could not check for updates using `pip` commands.  You should check manually."
msgstr "Impossibile controllare gli aggiornamenti usando i comandi `pip`. Dovresti controllare manualmente."

#: run.py:618
#, python-format
msgid "You can run a guided update by using the command:\n"
"    %s ./update.py"
msgstr "È possibile eseguire un aggiornamento guidato utilizzando il comando:\n"
"    %s ./update.py"

#: run.py:636
msgid "Value is above the maximum limit."
msgstr "Il valore è superiore al limite massimo."

#: run.py:638
msgid "Value must not be negative."
msgstr "Il valore non deve essere negativo."

#: run.py:642
#, python-format
msgid "Value for Max Logs Kept must be a number from 0 to %s"
msgstr "Valore per Max Logs Kept deve essere un numero da 0 a %s"

#: run.py:652
#, python-format
msgid "Log level '%s' is not available."
msgstr "Il livello di log '%s' non è disponibile."

#: run.py:656
#, python-format
msgid "Log Level must be one of:  %s"
msgstr "Il livello di log deve essere uno di:  %s"

#: run.py:666
msgid "Launch a music playing discord bot built using discord.py, youtubeDL, and ffmpeg."
msgstr "Avviare un bot discord per la visualizzazione musicale costruito utilizzando discord.py, youtubeDL, e ffmpeg."

#: run.py:669
msgid "Available via Github:"
msgstr "Disponibile tramite Github:"

#: run.py:673
msgid "For more help and support with this bot, join our discord:"
msgstr "Per più aiuto e supporto con questo bot, unisciti alla nostra discorda:"

#: run.py:675
msgid "This software is provided under the MIT License."
msgstr "Questo software è fornito sotto la licenza MIT."

#: run.py:677
msgid "See the `LICENSE` text file for complete details."
msgstr "Vedere il file di testo `LICENSE` per dettagli completi."

#: run.py:689
msgid "Override the default / system detected language for all text in MusicBot."
msgstr "Sovrascrivi la lingua predefinita / rilevata dal sistema per tutto il testo in MusicBot."

#: run.py:698
msgid "Use this language for all server-side log messages from MusicBot."
msgstr "Usa questa lingua per tutti i messaggi di registro del server da MusicBot."

#: run.py:707
msgid "Use this language for all messages sent to discord from MusicBot.\n"
"This does not prevent per-guild language selection."
msgstr "Usa questa lingua per tutti i messaggi inviati a discord da MusicBot.\n"
"Questo non impedisce la selezione della lingua per gilda."

#: run.py:718
msgid "Print the MusicBot version information and exit."
msgstr "Stampare le informazioni sulla versione di MusicBot ed uscire."

#: run.py:726
msgid "Skip all optional startup checks, including the update check."
msgstr "Salta tutti i controlli di avvio opzionali, incluso il controllo di aggiornamento."

#: run.py:734
msgid "Skip only the disk space check at startup."
msgstr "Salta solo il controllo dello spazio su disco all'avvio."

#: run.py:742
msgid "Skip only the update check at startup."
msgstr "Salta solo il controllo di aggiornamento all'avvio."

#: run.py:751
msgid "Disable MusicBot from trying to install dependencies when it cannot import them."
msgstr "Disabilita MusicBot dal tentativo di installare le dipendenze quando non può importarle."

#: run.py:762
#, python-format
msgid "Specify how many log files to keep, between 0 and %s inclusive. (Default: %s)"
msgstr "Specifica quanti file di registro mantenere, tra 0 e %s inclusi. (Predefinito: %s)"

#: run.py:772
#, python-format
msgid "Override the log level settings set in config. Must be one of: %s"
msgstr "Sovrascrivi le impostazioni del livello di log impostate nella configurazione. Deve essere una di: %s"

#: run.py:783
#, python-format
msgid "Override the default date format used when rotating log files. This should contain values compatible with strftime().  (Default:  '%s')"
msgstr "Sovrascrivi il formato di data predefinito usato per ruotare i file di registro. Questo dovrebbe contenere valori compatibili con strftime(). (Predefinito: '%s')"

#: run.py:798
#, python-format
msgid "Version:  %s"
msgstr "Versione:  %s"

#: run.py:873
msgid "Opened a new MusicBot instance. This terminal can be safely closed!"
msgstr "Apri una nuova istanza di MusicBot. Questo terminale può essere chiuso in sicurezza!"

#: run.py:944
#, python-format
msgid "Loading MusicBot version:  %s"
msgstr "Caricamento versione MusicBot:  %s"

#: run.py:945
#, python-format
msgid "Log opened:  %s"
msgstr "Log aperto:  %s"

#: run.py:946
#, python-format
msgid "Python version:  %s"
msgstr "Python version:  %s"

#: run.py:955
#, python-format
msgid "Changing working directory to:  %s"
msgstr "Modifica della directory di lavoro in:  %s"

#: run.py:959
msgid "Cannot start the bot!  You started `run.py` in the wrong directory and we could not locate `musicbot` and `.git` folders to verify a new directory location."
msgstr "Impossibile avviare il bot! Hai iniziato `run.py` nella directory sbagliata e non abbiamo potuto individuare `musicbot` e `. it` cartelle per verificare una nuova posizione della directory."

#: run.py:964
msgid "For best results, start `run.py` from the same folder you cloned MusicBot into.\n"
"If you did not use git to clone the repository, you are strongly urged to."
msgstr "Per ottenere risultati migliori, avvia `run.py` dalla stessa cartella in cui hai clonato MusicBot.\n"
"Se non hai usato git per clonare il repository, sei fortemente sollecitato."

#: run.py:1018
msgid "Certificate error is not a verification error, not trying certifi and exiting."
msgstr "L'errore del certificato non è un errore di verifica, non è un tentativo di certifi e di uscita."

#: run.py:1020
msgid "Here is the exact error, it could be a bug."
msgstr "Ecco l'errore esatto, potrebbe essere un bug."

#: run.py:1036
msgid "To easily add a certificate to Windows trust store, \n"
"you can open the failing site in Microsoft Edge or IE...\n"
msgstr "Per aggiungere facilmente un certificato all'archivio di fiducia di Windows, \n"
"è possibile aprire il sito fallito in Microsoft Edge o IE...\n"

#: run.py:1042
msgid "Could not get Issuer Certificate from default trust store, trying certifi instead."
msgstr "Impossibile ottenere il certificato dell'emittente dall'archivio di fiducia predefinito, tentando invece il certificifi."

#: run.py:1051
msgid "Syntax error (modification detected, did you edit the code?)"
msgstr "Errore di sintassi (modifica rilevata, hai modificato il codice?)"

#: run.py:1054
msgid "Syntax error (this is a bug, not your fault)"
msgstr "Errore di sintassi (questo è un bug, non colpa tua)"

#: run.py:1066
msgid "Cannot start MusicBot due to an error!\n\n"
"Problem:\n"
"  There was an error importing MusicBot or a dependency package.\n\n"
"Solution:\n"
"  You need to manually install pip packages for MusicBot\n"
"  or launch without `--no-install-deps` and MusicBot will try to install them for you."
msgstr "Impossibile avviare MusicBot a causa di un errore!\n\n"
"Problema:\n"
"  Si è verificato un errore durante l'importazione di MusicBot o di un pacchetto di dipendenze.\n\n"
"Soluzione:\n"
"  Devi installare manualmente pacchetti pip per MusicBot\n"
"  o lanciare senza `--no-install-deps` e MusicBot proverà ad installarli per te."

#: run.py:1099
msgid "Attempting to install MusicBot dependency packages automatically...\n"
msgstr "Tentativo di installare automaticamente i pacchetti di dipendenza di MusicBot...\n"

#: run.py:1108
#, python-format
msgid "MusicBot dependencies may not be installed!\n\n"
"Problem:\n"
"  The pip install process ended with a possible error.\n"
"  Some or all of the the dependencies may be missing.\n\n"
"Solution:\n"
"  You must manually install dependency packages.\n"
"  Open a CMD prompt / terminal to the MusicBot directory.\n"
"  You can try using the update scripts install packages.\n"
"  Or try this manual command:\n"
"    %(py_bin)s -m pip install -U -r ./requirements.txt\n\n"
"You can also ask for help in MusicBot's support discord:\n"
"  https://discord.gg/bots"
msgstr "Le dipendenze di MusicBot potrebbero non essere installate!\n\n"
"Problema:\n"
"  Il processo di installazione di pip è terminato con un possibile errore.\n"
"  Alcune o tutte le dipendenze potrebbero mancare.\n\n"
"Soluzione:\n"
"  È necessario installare manualmente i pacchetti di dipendenza.\n"
"  Aprire un prompt / terminale CMD nella directory di MusicBot.\n"
"  Puoi provare a utilizzare i pacchetti di installazione degli script di aggiornamento.\n"
"  O prova questo comando manuale:\n"
"    %(py_bin)s -m pip install -U -r . requirements.txt\n\n"
"Puoi anche chiedere aiuto nella discordia di supporto di MusicBot:\n"
"  https://discord.gg/bots"

#: run.py:1129
msgid "OK, lets hope installing dependencies worked!"
msgstr "OK, lascia funzionare la speranza di installare dipendenze!"

#: run.py:1141
msgid "MusicBot got an ImportError after trying to install packages. MusicBot must exit..."
msgstr "MusicBot ha ottenuto un ImportError dopo aver provato a installare i pacchetti. MusicBot deve uscire..."

#: run.py:1143
msgid "The exception which caused the above error: "
msgstr "L'eccezione che ha causato l'errore sopra: "

#: run.py:1159
msgid "MusicBot is doing a soft restart..."
msgstr "MusicBot sta facendo un riavvio morbido..."

#: run.py:1163
msgid "MusicBot is doing a full process restart..."
msgstr "MusicBot sta facendo un riavvio completo..."

#: run.py:1169
msgid "Error starting bot"
msgstr "Errore nell'avvio del bot"

#: run.py:1174
msgid "Closing event loop."
msgstr "Ciclo dell'evento di chiusura."

#: run.py:1179
#, python-format
msgid "Restarting in %s seconds..."
msgstr "Riavvio in %s secondi..."

#: run.py:1183
msgid "All done."
msgstr "Tutto fatto."

#: run.py:1212
msgid "OK, we're closing!"
msgstr "OK, stiamo chiudendo!"

